
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://kbase.github.io/cdm-schema/">
      
      
      
        <link rel="next" href="pages/about/">
      
      
        
      
      
      <link rel="icon" href="assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.1">
    
    
      
        <title>cdm-schema</title>
      
    
    
      <link rel="stylesheet" href="assets/stylesheets/main.484c7ddc.min.css">
      
        
        <link rel="stylesheet" href="assets/stylesheets/palette.ab4e12ef.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL(".",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#cdm_schema" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="." title="cdm-schema" class="md-header__button md-logo" aria-label="cdm-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            cdm-schema
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Index
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/kbase/cdm-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="." title="cdm-schema" class="md-nav__button md-logo" aria-label="cdm-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    cdm-schema
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/kbase/cdm-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    
  
    Index
  

    
  </span>
  
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="." class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    
  
    Index
  

    
  </span>
  
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      
        Classes
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      
        Slots
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      
        Enumerations
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      
        Types
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      
        Subsets
      
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="pages/about/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    About
  

    
  </span>
  
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      
        Classes
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      
        Slots
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      
        Enumerations
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      
        Types
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      
        Subsets
      
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="cdm_schema">cdm_schema</h1>
<p>Schema for KBase CDM</p>
<p>URI: http://kbase.github.io/cdm-schema/linkml/cdm_schema</p>
<p>Name: cdm_schema</p>
<h2 id="classes">Classes</h2>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Any/">Any</a></td>
<td>Used as a range for slots that have more than one possible type.</td>
</tr>
<tr>
<td><a href="AttributeMixin/">AttributeMixin</a></td>
<td>The attribute in an attribute-value pair. One of <code>attribute_cv_id</code>, <code>attribute_cv_label</code>, and <code>attribute_string</code> is required.</td>
</tr>
<tr>
<td><a href="AttributeValue/">AttributeValue</a></td>
<td>The value for any value of attribute for an entity. This object can hold both the un-normalized atomic value and the structured value.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ControlledTermValue/">ControlledTermValue</a></td>
<td>A quality, described using a text string from a controlled vocabulary or enum.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ControlledVocabularyTermValue/">ControlledVocabularyTermValue</a></td>
<td>A quality, described using a term from an ontology or schema with a stable persistent identifier.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="DateTimeValue/">DateTimeValue</a></td>
<td>A date or date and time value.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Geolocation/">Geolocation</a></td>
<td>A normalized value for a location on the earth's surface. Should be expressed in decimal degrees.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="QuantityRangeValue/">QuantityRangeValue</a></td>
<td>A numerical range, e.g. 5-7 cm.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="QuantityValue/">QuantityValue</a></td>
<td>A simple quantity, e.g. 2 cm.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="TextValue/">TextValue</a></td>
<td>A basic string value.</td>
</tr>
<tr>
<td><a href="EntityAttributeValue/">EntityAttributeValue</a></td>
<td>Class comprising all possible entity-attribute-value slots.</td>
</tr>
<tr>
<td><a href="EntityMixin/">EntityMixin</a></td>
<td>A generic class for capturing attribute-value information about an entity in a structured form.</td>
</tr>
<tr>
<td><a href="Schema/">Schema</a></td>
<td>The root class for the CDM schema.</td>
</tr>
<tr>
<td><a href="Table/">Table</a></td>
<td>Abstract class representing a table in the CDM schema.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Association/">Association</a></td>
<td>An association between an object--typically an entity such as a protein or a feature--and a classification system or ontology, such as the Gene Ontology, the Enzyme Classification, or TIGRFAMS domains.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Cluster/">Cluster</a></td>
<td>Represents an individual execution of a clustering protocol. See the ClusterMember class for clustering results.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ClusterMember/">ClusterMember</a></td>
<td>Relationship representing membership of a cluster. An optional score can be assigned to each cluster member.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Contig/">Contig</a></td>
<td>A contig (derived from the word "contiguous") is a set of DNA segments or sequences that overlap in a way that provides a contiguous representation of a genomic region. A contig should not contain any gaps.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigCollection/">ContigCollection</a></td>
<td>A set of individual, overlapping contigs that represent the complete sequenced genome of an organism.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Contributor/">Contributor</a></td>
<td>Represents a contributor to a resource.<br><br>Contributors must have a 'contributor_type', either 'Person' or 'Organization', and one of the 'name' fields: either 'given_name' and 'family_name' (for a person), or 'name' (for an organization or a person).<br><br>The 'contributor_role' field takes values from the DataCite and CRediT contributor roles vocabularies. For more information on these resources and choosing appropriate roles, please see the following links:<br><br>DataCite contributor roles: https://support.datacite.org/docs/datacite-metadata-schema-v44-recommended-and-optional-properties#7a-contributortype<br><br>CRediT contributor role taxonomy: https://credit.niso.org</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContributorXRoleXProject/">ContributorXRoleXProject</a></td>
<td>Describes the participation of a contributor in a project; ideally the contributor_role field will capture how the contributor was involved.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="DataSource/">DataSource</a></td>
<td>The source for a resource, dataset, association, etc.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="DataSourceNew/">DataSourceNew</a></td>
<td>The source dataset from which data within the CDM was extracted. This might be an API query; a set of files downloaded from a website or uploaded by a user; a database dump; etc. A given data source should have either version information (e.g. a release number, like those used by UniProt or RefSeq) or an access date to allow the original raw data dump to be recapitulated.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="EncodedFeature/">EncodedFeature</a></td>
<td>An entity generated from a feature, such as a transcript.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="EntailedEdge/">EntailedEdge</a></td>
<td>A relation graph edge that is inferred. This table contains links between the nodes that appear as subjects in the <a href="Statement">Statement</a> table. The graph contains all possible links between nodes and is created using a reasoner such as <a href="https://github.com/balhoff/relation-graph">relation-graph</a> to materialise inferrred links.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Entity/">Entity</a></td>
<td>An entity in the CDM.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Event/">Event</a></td>
<td>Something that happened.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Experiment/">Experiment</a></td>
<td>A discrete scientific procedure undertaken to make a discovery, test a hypothesis, or demonstrate a known fact. The protocol_id links to the workflow followed to perform the experiment.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ExperimentCondition/">ExperimentCondition</a></td>
<td>A measurement, reagent, or description of one aspect of the environment used in an experiment; examples include temperature; aerobic or anaerobic conditions; presence of a chemical in the environment. Used to describe the context, conditions, or set-up of an experiment.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ExperimentConditionSet/">ExperimentConditionSet</a></td>
<td>A unique combination of experimental conditions and entities that are used in a specific experiment. One experiment condition set is expected to comprise of multiple ExperimentConditions.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Feature/">Feature</a></td>
<td>A feature localized to an interval along a contig.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="FundingReference/">FundingReference</a></td>
<td>Represents a funding source for a resource, including the funding body and the grant awarded.<br><br>One (or more) of the fields <code>grant_id</code>, <code>grant_url</code>, or <code>funder.organization_name</code> is required; others are optional.<br><br>Recommended resources for organization identifiers include:<br>  - Research Organization Registry, http://ror.org<br>  - International Standard Name Identifier, https://isni.org<br>  - Crossref Funder Registry, https://www.crossref.org/services/funder-registry/ (to be subsumed into ROR)<br><br>Some organizations may have a digital object identifier (DOI).</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="GoldEnvironmentalContext/">GoldEnvironmentalContext</a></td>
<td>Environmental context, described using JGI's five level system.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Identifier/">Identifier</a></td>
<td>A string used as a resolvable (external) identifier for an entity. This should be a CURIE in the form <code>&lt;database_prefix&gt;:&lt;local_identifier&gt;</code>. <a href="https://bioregistry.io">Bioregistry</a> is used as the canonical reference for CURIE database prefixes; please use the prefix exactly as written in the Bioregistry entry.<br><br>If the string cannot be resolved to an URL, it should be added to the <code>Name</code> table instead.<br><br>This table is used for capturing external IDs. The internal CDM identifier should be used in the *_id field (e.g. feature_id, protein_id, contig_collection_id).</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="License/">License</a></td>
<td>License information for the resource.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="LinkerTable/">LinkerTable</a></td>
<td>Tables for linking between tables.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="AssociationXSupportingObject/">AssociationXSupportingObject</a></td>
<td>Links associations to entities to capture supporting objects in an association. May be a biological entity, such as a protein or feature, or a URL to a resource (e.g. a publication) that supports the association. Where possible, CDM identifiers should be used.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigXContigCollection/">ContigXContigCollection</a></td>
<td>Captures the relationship between a contig and a contig collection; equivalent to contig part-of contig collection.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigXEncodedFeature/">ContigXEncodedFeature</a></td>
<td>Captures the relationship between a contig and an encoded feature.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigXFeature/">ContigXFeature</a></td>
<td>Captures the relationship between a contig and a feature; equivalent to feature part-of contig.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigXProtein/">ContigXProtein</a></td>
<td>Captures the relationship between a contig and a protein; equivalent to protein is ribosomal translation of (http://purl.obolibrary.org/obo/RO_0002512) contig.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigCollectionXEncodedFeature/">ContigCollectionXEncodedFeature</a></td>
<td>Captures the relationship between a contig collection and an encoded feature.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigCollectionXFeature/">ContigCollectionXFeature</a></td>
<td>Captures the relationship between a contig collection and a feature; equivalent to feature part-of contig collection.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigCollectionXProtein/">ContigCollectionXProtein</a></td>
<td>Captures the relationship between a contig collection and a protein; equivalent to protein is ribosomal translation of (http://purl.obolibrary.org/obo/RO_0002512) contig collection.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContributorXDataSource/">ContributorXDataSource</a></td>
<td>Captures the people and/or organizations involved in producing a dataset; ideally the contributor_role field will capture how the contributor was involved.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContributorAffiliation/">ContributorAffiliation</a></td>
<td>Captures relationships between contributors where one contributor is part of another contributor, e.g. a member of a group or a group that is part of a larger organization.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="DataSourceXDescription/">DataSourceXDescription</a></td>
<td>Links a data source to a description (e.g. the abstract or a free text description).</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="DataSourceXFundingReference/">DataSourceXFundingReference</a></td>
<td>Links a data source to a funding reference.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="DataSourceXLicense/">DataSourceXLicense</a></td>
<td>Links a data source to a license.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="DataSourceXTitle/">DataSourceXTitle</a></td>
<td>Links a data source to a title.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="EncodedFeatureXFeature/">EncodedFeatureXFeature</a></td>
<td>Captures the relationship between a feature and its transcription product.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="EncodedFeatureXProtein/">EncodedFeatureXProtein</a></td>
<td>Captures the relationship between an encoded feature (RNA of some sort) and a protein.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="EntityIdentifiers/">EntityIdentifiers</a></td>
<td>Represents the link between an entity and its identifiers.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="EntityNames/">EntityNames</a></td>
<td>Represents the link between an entity and its names.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="FeatureXProtein/">FeatureXProtein</a></td>
<td>Captures the relationship between a feature and a protein; equivalent to feature encodes protein.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ProtocolVariable/">ProtocolVariable</a></td>
<td>A variable that may or may not be set as part of an experiment.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Measurement/">Measurement</a></td>
<td>The value of a specified variable_id under the specified conditions.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="MeasurementSet/">MeasurementSet</a></td>
<td>Grouping table to collate a set of protocol outputs by variable, quality, and timestamp.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="MixsEnvironmentalContext/">MixsEnvironmentalContext</a></td>
<td>Environmental context, described using the MiXS convention of broad and local environment, plus the medium.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Name/">Name</a></td>
<td>A string used as the name or label for an entity. This may be a primary name, alternative name, synonym, acronym, or any other label used to refer to an entity.<br><br>Identifiers that look like CURIEs or database references, but which cannot be resolved using <a href="https://bioregistry.io">Bioregistry</a> or <a href="https://identifiers.org">identifiers.org</a> should be added to the <code>Name</code> table.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="OrderedProtocolStep/">OrderedProtocolStep</a></td>
<td>A list of the steps in a protocol; the step_index indicates the order in which they should be executed.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Parameter/">Parameter</a></td>
<td>A parameter in a protocol. Currently specific to computational protocols.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Prefix/">Prefix</a></td>
<td>Maps CURIEs to URIs</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Project/">Project</a></td>
<td>Administrative unit for collecting data related to a certain topic, location, data type, grant funding, and so on.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Protein/">Protein</a></td>
<td>Proteins are large, complex molecules made up of one or more long, folded chains of amino acids, whose sequences are determined by the DNA sequence of the protein-encoding gene.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Protocol/">Protocol</a></td>
<td>A defined method or set of methods.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ProtocolExecution/">ProtocolExecution</a></td>
<td>An instance of executing a protocol. Used for</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ProtocolInput/">ProtocolInput</a></td>
<td>An input parameter for a protocol.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ProtocolInputSet/">ProtocolInputSet</a></td>
<td>A set of input parameters for a protocol.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ProtocolOutput/">ProtocolOutput</a></td>
<td>The output of a protocol.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ProtocolStep/">ProtocolStep</a></td>
<td>A step in a protocol.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Publication/">Publication</a></td>
<td>A publication (e.g. journal article).</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ResourceDescription/">ResourceDescription</a></td>
<td>Textual information about the resource being represented.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ResourceTitle/">ResourceTitle</a></td>
<td>Represents the title or name of a resource, the type of that title, and the language used (if appropriate).<br><br>The <code>title</code> field is required; <code>title_type</code> is only necessary if the text is not the primary title.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Sample/">Sample</a></td>
<td>A material entity that can be characterised by an experiment.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Sequence/">Sequence</a></td>
<td>A sequence of nucleotides or amino acids.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Statement/">Statement</a></td>
<td>Represents an RDF triple, a statement in the form "<em>subject</em> <em>predicate</em> <em>object</em>" or "<em>subject</em> <em>predicate</em> <em>value</em>".<br><br>See <a href="https://incatools.github.io/semantic-sql/">Semantic SQL</a> for more information on the contents of this table and how it is populated.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Variable/">Variable</a></td>
<td>A variable (input, output, environmental, etc.) that can be set or measured as part of a protocol.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="VariableValue/">VariableValue</a></td>
<td>The possible types for the value of a variable. Should be a LinkML data type or one of the defined CDM data types.</td>
</tr>
<tr>
<td><a href="UnitMixin/">UnitMixin</a></td>
<td>The unit used in expressing a quantity value.</td>
</tr>
</tbody>
</table>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Slot</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="affiliation_id/">affiliation_id</a></td>
<td>The ID of the organization to which a contributor belongs. Should be the ID of another contributor.</td>
</tr>
<tr>
<td><a href="aggregator_knowledge_source/">aggregator_knowledge_source</a></td>
<td>The knowledge source that aggregated the association. Should be a CDM ID from the Contributor or DataSource table.</td>
</tr>
<tr>
<td><a href="annotation_date/">annotation_date</a></td>
<td>The date when the annotation was made.</td>
</tr>
<tr>
<td><a href="asm_score/">asm_score</a></td>
<td>A composite score for comparing contig collection quality.</td>
</tr>
<tr>
<td><a href="association_id/">association_id</a></td>
<td>Internal (CDM) unique identifier for an association.</td>
</tr>
<tr>
<td><a href="association_x_supporting_objects/">association_x_supporting_objects</a></td>
<td>All association x supporting object records in the schema.</td>
</tr>
<tr>
<td><a href="associations/">associations</a></td>
<td>All associations in the schema.</td>
</tr>
<tr>
<td><a href="attribute_cv_id/">attribute_cv_id</a></td>
<td>The attribute being represented. For attributes that are in a controlled vocabulary, ontology, or enumeration, this attribute should capture the term ID from the controlled vocabulary.</td>
</tr>
<tr>
<td><a href="attribute_cv_label/">attribute_cv_label</a></td>
<td>The attribute being represented. For attributes that are in a controlled vocabulary, ontology, or enumeration, this attribute should capture the term from the controlled vocabulary.</td>
</tr>
<tr>
<td><a href="attribute_string/">attribute_string</a></td>
<td>The attribute being represented, as a text string. This field should only be used if the attribute is not represented in a controlled vocabulary, ontology, or enumeration.</td>
</tr>
<tr>
<td><a href="base/">base</a></td>
<td>The base URI a prefix will expand to.</td>
</tr>
<tr>
<td><a href="cardinality/">cardinality</a></td>
<td>The cardinality of the parameter.</td>
</tr>
<tr>
<td><a href="cds_phase/">cds_phase</a></td>
<td>For features of type CDS, the phase indicates where the next codon begins relative to the 5' end (where the 5' end of the CDS is relative to the strand of the CDS feature) of the current CDS feature. cds_phase is required if the feature type is CDS.</td>
</tr>
<tr>
<td><a href="checkm_completeness/">checkm_completeness</a></td>
<td>Estimate of the completeness of a contig collection (MAG or genome), estimated by CheckM tool. Ensure that percentage values are converted to floats.</td>
</tr>
<tr>
<td><a href="checkm_contamination/">checkm_contamination</a></td>
<td>Estimate of the contamination of a contig collection (MAG or genome), estimated by CheckM tool. Ensure that percentage values are converted to floats.</td>
</tr>
<tr>
<td><a href="checkm_version/">checkm_version</a></td>
<td>Version of the CheckM tool used.</td>
</tr>
<tr>
<td><a href="checksum/">checksum</a></td>
<td>The checksum of the sequence, used to verify its integrity.</td>
</tr>
<tr>
<td><a href="cluster_id/">cluster_id</a></td>
<td>Internal (CDM) unique identifier for a cluster.<br>From the Entity table: entity_id where entity_type == 'Cluster'.</td>
</tr>
<tr>
<td><a href="cluster_members/">cluster_members</a></td>
<td>All cluster members in the schema.</td>
</tr>
<tr>
<td><a href="clusters/">clusters</a></td>
<td>All clusters in the schema.</td>
</tr>
<tr>
<td><a href="comments/">comments</a></td>
<td>Any comments about the association.</td>
</tr>
<tr>
<td><a href="contig_bp/">contig_bp</a></td>
<td>Total size in bp of all contigs</td>
</tr>
<tr>
<td><a href="contig_collection_id/">contig_collection_id</a></td>
<td>Internal (CDM) unique identifier for a contig collection.<br>From the Entity table: entity_id where entity_type == 'ContigCollection'.</td>
</tr>
<tr>
<td><a href="contig_collection_type/">contig_collection_type</a></td>
<td>The type of contig collection.</td>
</tr>
<tr>
<td><a href="contig_collection_x_encoded_features/">contig_collection_x_encoded_features</a></td>
<td>All contig collection x encoded feature records in the schema.</td>
</tr>
<tr>
<td><a href="contig_collection_x_features/">contig_collection_x_features</a></td>
<td>All contig collection x feature records in the schema.</td>
</tr>
<tr>
<td><a href="contig_collection_x_proteins/">contig_collection_x_proteins</a></td>
<td>All contig collection x protein records in the schema.</td>
</tr>
<tr>
<td><a href="contig_collections/">contig_collections</a></td>
<td>All contig collections in the schema.</td>
</tr>
<tr>
<td><a href="contig_id/">contig_id</a></td>
<td>Internal (CDM) unique identifier for a contig.<br>From the Entity table: entity_id where entity_type == 'Contig'.</td>
</tr>
<tr>
<td><a href="contig_l50/">contig_l50</a></td>
<td>Given a set of contigs, the L50 is defined as the sequence length of the shortest contig at 50% of the total contig collection length</td>
</tr>
<tr>
<td><a href="contig_l90/">contig_l90</a></td>
<td>The L90 statistic is less than or equal to the L50 statistic; it is the length for which the collection of all contigs of that length or longer contains at least 90% of the sum of the lengths of all contigs</td>
</tr>
<tr>
<td><a href="contig_logsum/">contig_logsum</a></td>
<td>The sum of the (length*log(length)) of all contigs, times some constant.</td>
</tr>
<tr>
<td><a href="contig_max/">contig_max</a></td>
<td>Maximum contig length</td>
</tr>
<tr>
<td><a href="contig_n50/">contig_n50</a></td>
<td>Given a set of contigs, each with its own length, the N50 count is defined as the smallest number_of_contigs whose length sum makes up half of contig collection size</td>
</tr>
<tr>
<td><a href="contig_n90/">contig_n90</a></td>
<td>Given a set of contigs, each with its own length, the N90 count is defined as the smallest number of contigs whose length sum makes up 90% of contig collection size</td>
</tr>
<tr>
<td><a href="contig_powersum/">contig_powersum</a></td>
<td>Powersum of all contigs is the same as logsum except that it uses the sum of (length*(length^P)) for some power P (default P=0.25)</td>
</tr>
<tr>
<td><a href="contig_x_contig_collections/">contig_x_contig_collections</a></td>
<td>All contig x contig collection records in the schema.</td>
</tr>
<tr>
<td><a href="contig_x_encoded_features/">contig_x_encoded_features</a></td>
<td>All contig x encoded feature records in the schema.</td>
</tr>
<tr>
<td><a href="contig_x_features/">contig_x_features</a></td>
<td>All contig x feature records in the schema.</td>
</tr>
<tr>
<td><a href="contig_x_proteins/">contig_x_proteins</a></td>
<td>All contig x protein records in the schema.</td>
</tr>
<tr>
<td><a href="contigs/">contigs</a></td>
<td>All contigs in the schema.</td>
</tr>
<tr>
<td><a href="contributor_affiliations/">contributor_affiliations</a></td>
<td>All contributor affiliations in the schema.</td>
</tr>
<tr>
<td><a href="contributor_id/">contributor_id</a></td>
<td>Internal (CDM) unique identifier for a contributor.<br>From the Entity table: entity_id where entity_type == 'Contributor'.</td>
</tr>
<tr>
<td><a href="contributor_role/">contributor_role</a></td>
<td>Role(s) played by the contributor when working on the experiment. If more than one role was played, additional rows should be added to represent each role.</td>
</tr>
<tr>
<td><a href="contributor_type/">contributor_type</a></td>
<td>Must be either 'Person' or 'Organization'.</td>
</tr>
<tr>
<td><a href="contributor_x_role_x_project/">contributor_x_role_x_project</a></td>
<td>All contributor x role x project records in the schema.</td>
</tr>
<tr>
<td><a href="contributors/">contributors</a></td>
<td>All contributors in the schema.</td>
</tr>
<tr>
<td><a href="created/">created</a></td>
<td>Date/timestamp for when the entity was created or added to the CDM.</td>
</tr>
<tr>
<td><a href="created_at/">created_at</a></td>
<td>The time at which the event started or was created.</td>
</tr>
<tr>
<td><a href="data_source_created/">data_source_created</a></td>
<td>Date/timestamp for when the entity was created or added to the data source.</td>
</tr>
<tr>
<td><a href="data_source_entity_id/">data_source_entity_id</a></td>
<td>The primary, ideally unique, ID of the entity at the data source.</td>
</tr>
<tr>
<td><a href="data_source_id/">data_source_id</a></td>
<td>Internal (CDM) unique identifier for a data source.<br>From the Entity table: entity_id where entity_type == 'DataSource'.</td>
</tr>
<tr>
<td><a href="data_source_updated/">data_source_updated</a></td>
<td>Date/timestamp for when the entity was updated in the data source.</td>
</tr>
<tr>
<td><a href="data_source_x_descriptions/">data_source_x_descriptions</a></td>
<td>All data source descriptions in the schema.</td>
</tr>
<tr>
<td><a href="data_source_x_funding_references/">data_source_x_funding_references</a></td>
<td>All data source x funding reference records in the schema.</td>
</tr>
<tr>
<td><a href="data_source_x_licenses/">data_source_x_licenses</a></td>
<td>All data source x license records in the schema.</td>
</tr>
<tr>
<td><a href="data_source_x_titles/">data_source_x_titles</a></td>
<td>All data source x title records in the schema.</td>
</tr>
<tr>
<td><a href="data_sources/">data_sources</a></td>
<td>All data sources in the schema.</td>
</tr>
<tr>
<td><a href="datatype/">datatype</a></td>
<td>the rdf datatype of the value, for example, xsd:string or xsd:float.</td>
</tr>
<tr>
<td><a href="date_accessed/">date_accessed</a></td>
<td>The date when the data was downloaded from the data source.</td>
</tr>
<tr>
<td><a href="date_published/">date_published</a></td>
<td>The date when the data source was originally made public.</td>
</tr>
<tr>
<td><a href="date_time/">date_time</a></td>
<td>A date or date and time, expressed in ISO 8601 format with timezone indicators where appropriate.  The date or date/time value, expressed in ISO 8601-compatible form. Dates should be expressed as YYYY-MM-DD; times should be expressed as HH:MM:SS with optional milliseconds and an indication of the timezone.</td>
</tr>
<tr>
<td><a href="date_updated/">date_updated</a></td>
<td>The date when the data source was last updated.</td>
</tr>
<tr>
<td><a href="default/">default</a></td>
<td>The default value for the parameter if a value is not supplied.</td>
</tr>
<tr>
<td><a href="description/">description</a></td>
<td>Brief textual definition or description.</td>
</tr>
<tr>
<td><a href="description_text/">description_text</a></td>
<td>The text content of the informational element.</td>
</tr>
<tr>
<td><a href="description_type/">description_type</a></td>
<td>The type of text being represented.</td>
</tr>
<tr>
<td><a href="doi/">doi</a></td>
<td>The DOI for a protocol.</td>
</tr>
<tr>
<td><a href="e_value/">e_value</a></td>
<td>The 'score' of the feature. The semantics of this field are ill-defined. E-values should be used for sequence similarity features.</td>
</tr>
<tr>
<td><a href="ecosystem/">ecosystem</a></td>
<td>JGI GOLD descriptor representing the top level ecosystem categorization.</td>
</tr>
<tr>
<td><a href="ecosystem_category/">ecosystem_category</a></td>
<td>JGI GOLD descriptor representing the ecosystem category.</td>
</tr>
<tr>
<td><a href="ecosystem_subtype/">ecosystem_subtype</a></td>
<td>JGI GOLD descriptor representing the subtype of ecosystem. May be "Unclassified".</td>
</tr>
<tr>
<td><a href="ecosystem_type/">ecosystem_type</a></td>
<td>JGI GOLD descriptor representing the ecosystem type. May be "Unclassified".</td>
</tr>
<tr>
<td><a href="encoded_feature_id/">encoded_feature_id</a></td>
<td>Internal (CDM) unique identifier for an encoded feature.<br>From the Entity table: entity_id where entity_type == 'EncodedFeature'.</td>
</tr>
<tr>
<td><a href="encoded_feature_x_features/">encoded_feature_x_features</a></td>
<td>All encoded feature x feature records in the schema.</td>
</tr>
<tr>
<td><a href="encoded_feature_x_proteins/">encoded_feature_x_proteins</a></td>
<td>All encoded feature x protein records in the schema.</td>
</tr>
<tr>
<td><a href="encoded_features/">encoded_features</a></td>
<td>All encoded features in the schema.</td>
</tr>
<tr>
<td><a href="end/">end</a></td>
<td>The start and end coordinates of the feature are given in positive 1-based int coordinates, relative to the landmark given in column one. Start is always less than or equal to end. For features that cross the origin of a circular feature (e.g. most bacterial genomes, plasmids, and some viral genomes), the requirement for start to be less than or equal to end is satisfied by making end = the position of the end + the length of the landmark feature. For zero-length features, such as insertion sites, start equals end and the implied site is to the right of the indicated base in the direction of the landmark.</td>
</tr>
<tr>
<td><a href="entailed_edges/">entailed_edges</a></td>
<td>All entailed edges in the schema.</td>
</tr>
<tr>
<td><a href="entities/">entities</a></td>
<td>All entities in the schema.</td>
</tr>
<tr>
<td><a href="entity_attribute_values/">entity_attribute_values</a></td>
<td>All entity attribute values in the schema.</td>
</tr>
<tr>
<td><a href="entity_id/">entity_id</a></td>
<td>Internal (CDM) unique identifier for an entity.</td>
</tr>
<tr>
<td><a href="entity_identifiers/">entity_identifiers</a></td>
<td>All identifier x entity records in the schema.</td>
</tr>
<tr>
<td><a href="entity_names/">entity_names</a></td>
<td>All name x entity records in the schema.</td>
</tr>
<tr>
<td><a href="entity_type/">entity_type</a></td>
<td>Type of entity being clustered.</td>
</tr>
<tr>
<td><a href="env_broad_scale/">env_broad_scale</a></td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO's biome class: http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
</tr>
<tr>
<td><a href="env_local_scale/">env_local_scale</a></td>
<td>Report the entity or entities which are in the sample or specimen's local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
</tr>
<tr>
<td><a href="env_medium/">env_medium</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
</tr>
<tr>
<td><a href="event_id/">event_id</a></td>
<td>Internal (CDM) unique identifier for an event.</td>
</tr>
<tr>
<td><a href="events/">events</a></td>
<td>All events in the schema.</td>
</tr>
<tr>
<td><a href="evidence_for_existence/">evidence_for_existence</a></td>
<td>The evidence that this protein exists. For example, the protein may have been isolated from a cell, or it may be predicted based on sequence features.</td>
</tr>
<tr>
<td><a href="evidence_type/">evidence_type</a></td>
<td>The type of evidence supporting the association. Should be a term from the Evidence and Conclusion Ontology (ECO). Specific pieces of evidence that support the association should be added as supporting objects, in the AssociationSupportingObject table.</td>
</tr>
<tr>
<td><a href="experiment_condition_id/">experiment_condition_id</a></td>
<td>Internal (CDM) unique identifier for an experiment condition.</td>
</tr>
<tr>
<td><a href="experiment_condition_set_id/">experiment_condition_set_id</a></td>
<td>Internal (CDM) unique identifier for a set of experimental conditions.</td>
</tr>
<tr>
<td><a href="experiment_condition_sets/">experiment_condition_sets</a></td>
<td>All experiment condition sets in the schema.</td>
</tr>
<tr>
<td><a href="experiment_conditions/">experiment_conditions</a></td>
<td>All experiment conditions in the schema.</td>
</tr>
<tr>
<td><a href="experiment_id/">experiment_id</a></td>
<td>Internal (CDM) unique identifier for an experiment.</td>
</tr>
<tr>
<td><a href="experiments/">experiments</a></td>
<td>All experiments in the schema.</td>
</tr>
<tr>
<td><a href="family_name/">family_name</a></td>
<td>The family name(s) of the contributor.</td>
</tr>
<tr>
<td><a href="feature_id/">feature_id</a></td>
<td>Internal (CDM) unique identifier for a feature.<br>From the Entity table: entity_id where entity_type == 'Feature'.</td>
</tr>
<tr>
<td><a href="feature_x_proteins/">feature_x_proteins</a></td>
<td>All feature x protein records in the schema.</td>
</tr>
<tr>
<td><a href="features/">features</a></td>
<td>All features in the schema.</td>
</tr>
<tr>
<td><a href="funder/">funder</a></td>
<td>The funder for the grant or award.</td>
</tr>
<tr>
<td><a href="funding_reference_id/">funding_reference_id</a></td>
<td>Internal (CDM) unique identifier for a specific source of funding -- e.g. a grant or award.<br>From the Entity table: entity_id where entity_type == 'FundingReference'.</td>
</tr>
<tr>
<td><a href="funding_references/">funding_references</a></td>
<td>All funding references in the schema.</td>
</tr>
<tr>
<td><a href="gap_percent/">gap_percent</a></td>
<td>The gap size percentage of all scaffolds</td>
</tr>
<tr>
<td><a href="gc_average/">gc_average</a></td>
<td>The average GC content of the contig collection, expressed as a percentage</td>
</tr>
<tr>
<td><a href="gc_content/">gc_content</a></td>
<td>GC content of the contig, expressed as a percentage.</td>
</tr>
<tr>
<td><a href="gc_std/">gc_std</a></td>
<td>The standard deviation of GC content across the contig collection</td>
</tr>
<tr>
<td><a href="given_name/">given_name</a></td>
<td>The given name(s) of the contributor.</td>
</tr>
<tr>
<td><a href="gold_environmental_context_id/">gold_environmental_context_id</a></td>
<td>Internal (CDM) unique identifier for a GOLD environmental context.</td>
</tr>
<tr>
<td><a href="gold_environmental_contexts/">gold_environmental_contexts</a></td>
<td>All GOLD environmental contexts in the schema.</td>
</tr>
<tr>
<td><a href="grant_id/">grant_id</a></td>
<td>Code for the grant, assigned by the funder.</td>
</tr>
<tr>
<td><a href="grant_title/">grant_title</a></td>
<td>Title for the grant.</td>
</tr>
<tr>
<td><a href="grant_url/">grant_url</a></td>
<td>URL for the grant.</td>
</tr>
<tr>
<td><a href="gtdb_taxon_id/">gtdb_taxon_id</a></td>
<td>The GTDB taxon ID for this contig collection.</td>
</tr>
<tr>
<td><a href="has_stop_codon/">has_stop_codon</a></td>
<td>Captures whether or not the sequence includes stop coordinates.</td>
</tr>
<tr>
<td><a href="hash/">hash</a></td>
<td>A hash value generated from one or more object attributes that serves to ensure the entity is unique.</td>
</tr>
<tr>
<td><a href="id/">id</a></td>
<td>An identifier for an element. Note blank node ids are not unique across databases</td>
</tr>
<tr>
<td><a href="identifier/">identifier</a></td>
<td>Fully-qualified URL or CURIE used as an identifier for an entity.</td>
</tr>
<tr>
<td><a href="identifiers/">identifiers</a></td>
<td>All identifiers in the schema.</td>
</tr>
<tr>
<td><a href="is_representative/">is_representative</a></td>
<td>Whether or not this member is the representative for the cluster. If 'is_representative' is false, it is assumed that this is a cluster member.</td>
</tr>
<tr>
<td><a href="is_seed/">is_seed</a></td>
<td>Whether or not this is the seed for this cluster.</td>
</tr>
<tr>
<td><a href="language/">language</a></td>
<td>the human language in which the value is encoded, e.g. 'en'</td>
</tr>
<tr>
<td><a href="latitude/">latitude</a></td>
<td>The latitude portion of a geolocation.</td>
</tr>
<tr>
<td><a href="length/">length</a></td>
<td>Length of the contig in bp.</td>
</tr>
<tr>
<td><a href="license/">license</a></td>
<td>Usage license for the resource. Use one of the SPDX license identifiers or provide a link to the license text if no SPDX ID is available.</td>
</tr>
<tr>
<td><a href="license_id/">license_id</a></td>
<td>Unique identifier for a license.</td>
</tr>
<tr>
<td><a href="licenses/">licenses</a></td>
<td>All licenses in the schema.</td>
</tr>
<tr>
<td><a href="location/">location</a></td>
<td>The location for this event. May be described in terms of coordinates.</td>
</tr>
<tr>
<td><a href="longitude/">longitude</a></td>
<td>The longitude portion of a geolocation.</td>
</tr>
<tr>
<td><a href="maximum_numeric_value/">maximum_numeric_value</a></td>
<td>The maximum value part, expressed as number, of the quantity value when the value covers a range.</td>
</tr>
<tr>
<td><a href="measurement_id/">measurement_id</a></td>
<td>Internal (CDM) unique identifier for a measurement.</td>
</tr>
<tr>
<td><a href="measurement_set_id/">measurement_set_id</a></td>
<td>Internal (CDM) unique identifier for a specified variable in a specified experiment.</td>
</tr>
<tr>
<td><a href="measurement_sets/">measurement_sets</a></td>
<td>All measurement sets in the schema.</td>
</tr>
<tr>
<td><a href="measurements/">measurements</a></td>
<td>All measurements in the schema.</td>
</tr>
<tr>
<td><a href="minimum_numeric_value/">minimum_numeric_value</a></td>
<td>The minimum value part, expressed as number, of the quantity value when the value covers a range.</td>
</tr>
<tr>
<td><a href="mixs_environmental_context_id/">mixs_environmental_context_id</a></td>
<td>Internal (CDM) unique identifier for a mixs environmental context.</td>
</tr>
<tr>
<td><a href="mixs_environmental_contexts/">mixs_environmental_contexts</a></td>
<td>All MIxS environmental contexts in the schema.</td>
</tr>
<tr>
<td><a href="n_chromosomes/">n_chromosomes</a></td>
<td>Total number of chromosomes</td>
</tr>
<tr>
<td><a href="n_contigs/">n_contigs</a></td>
<td>Total number of contigs</td>
</tr>
<tr>
<td><a href="n_scaffolds/">n_scaffolds</a></td>
<td>Total number of scaffolds</td>
</tr>
<tr>
<td><a href="name/">name</a></td>
<td>A string used as a name or title.</td>
</tr>
<tr>
<td><a href="name_cv_id/">name_cv_id</a></td>
<td>If the <code>name</code> is from a controlled vocabulary (CV), the curie of the controlled vocabulary term.</td>
</tr>
<tr>
<td><a href="names/">names</a></td>
<td>All names in the schema.</td>
</tr>
<tr>
<td><a href="ncbi_taxon_id/">ncbi_taxon_id</a></td>
<td>The NCBI taxon ID for this contig collection.</td>
</tr>
<tr>
<td><a href="negated/">negated</a></td>
<td>If true, the relationship between the subject and object is negated. For example, consider an association where the subject is a protein ID, the object is the GO term for "glucose biosynthesis", and the predicate is "involved in". With the "negated" field set to false, the association is interpreted as "<protein ID> is involved in glucose biosynthesis". With the "negated" field set to true, the association is interpreted as "<protein ID> is not involved in glucose biosynthesis".</td>
</tr>
<tr>
<td><a href="numeric_value/">numeric_value</a></td>
<td>The numerical part of a quantity value.</td>
</tr>
<tr>
<td><a href="object/">object</a></td>
<td>Note the range of this slot is always a node. If the triple represents a literal, the "value" field will be populated instead.</td>
</tr>
<tr>
<td><a href="ordered_protocol_steps/">ordered_protocol_steps</a></td>
<td>All ordered protocol steps in the schema.</td>
</tr>
<tr>
<td><a href="p_value/">p_value</a></td>
<td>The 'score' of the feature. The semantics of this field are ill-defined. P-values should be used for ab initio gene prediction features.</td>
</tr>
<tr>
<td><a href="parameter_id/">parameter_id</a></td>
<td>Internal (CDM) unique identifier for a parameter of a protocol.</td>
</tr>
<tr>
<td><a href="parameter_type/">parameter_type</a></td>
<td>Whether the parameter applies to the protocol input or output.</td>
</tr>
<tr>
<td><a href="parameters/">parameters</a></td>
<td>All parameters in the schema.</td>
</tr>
<tr>
<td><a href="predicate/">predicate</a></td>
<td>The predicate of the statement</td>
</tr>
<tr>
<td><a href="prefix/">prefix</a></td>
<td>A standardized prefix such as 'GO' or 'rdf' or 'FlyBase'.</td>
</tr>
<tr>
<td><a href="prefixes/">prefixes</a></td>
<td>The prefix mappings for the schema.</td>
</tr>
<tr>
<td><a href="primary_knowledge_source/">primary_knowledge_source</a></td>
<td>The knowledge source or contributor that created the association. Should be a CDM ID from the Contributor or DataSource table.</td>
</tr>
<tr>
<td><a href="project_id/">project_id</a></td>
<td>Internal (CDM) unique identifier for a project.<br>From the Entity table: entity_id where entity_type == 'Project'.</td>
</tr>
<tr>
<td><a href="projects/">projects</a></td>
<td>All projects in the schema.</td>
</tr>
<tr>
<td><a href="protein_id/">protein_id</a></td>
<td>Internal (CDM) unique identifier for a protein.<br>From the Entity table: entity_id where entity_type == 'Protein'.</td>
</tr>
<tr>
<td><a href="proteins/">proteins</a></td>
<td>All proteins in the schema.</td>
</tr>
<tr>
<td><a href="protocol_execution_id/">protocol_execution_id</a></td>
<td>Internal CDM unique identifier for an execution of a protocol.</td>
</tr>
<tr>
<td><a href="protocol_executions/">protocol_executions</a></td>
<td>All protocol executions in the schema.</td>
</tr>
<tr>
<td><a href="protocol_id/">protocol_id</a></td>
<td>Internal (CDM) unique identifier for a protocol.<br>From the Entity table: entity_id where entity_type == 'Protocol'.</td>
</tr>
<tr>
<td><a href="protocol_input_id/">protocol_input_id</a></td>
<td>Internal CDM unique identifier for the value of an input parameter for a protocol.</td>
</tr>
<tr>
<td><a href="protocol_input_set_id/">protocol_input_set_id</a></td>
<td>Internal CDM unique identifier for a set of input parameter values for a protocol.</td>
</tr>
<tr>
<td><a href="protocol_input_sets/">protocol_input_sets</a></td>
<td>All protocol input sets in the schema.</td>
</tr>
<tr>
<td><a href="protocol_inputs/">protocol_inputs</a></td>
<td>All protocol inputs in the schema.</td>
</tr>
<tr>
<td><a href="protocol_output_id/">protocol_output_id</a></td>
<td>Internal CDM unique identifier for the value of an output of a protocol.</td>
</tr>
<tr>
<td><a href="protocol_outputs/">protocol_outputs</a></td>
<td>All protocol outputs in the schema.</td>
</tr>
<tr>
<td><a href="protocol_step_id/">protocol_step_id</a></td>
<td>Internal CDM unique identifier for a step in a protocol.</td>
</tr>
<tr>
<td><a href="protocol_steps/">protocol_steps</a></td>
<td>All protocol steps in the schema.</td>
</tr>
<tr>
<td><a href="protocol_variable_id/">protocol_variable_id</a></td>
<td>Internal CDM unique identifier for a variable in a protocol.</td>
</tr>
<tr>
<td><a href="protocol_variables/">protocol_variables</a></td>
<td>All protocol variables in the schema.</td>
</tr>
<tr>
<td><a href="protocols/">protocols</a></td>
<td>All protocols in the schema.</td>
</tr>
<tr>
<td><a href="publication_id/">publication_id</a></td>
<td>Unique identifier for a publication - e.g. PMID, DOI, URL, etc.</td>
</tr>
<tr>
<td><a href="publications/">publications</a></td>
<td>All publications in the schema.</td>
</tr>
<tr>
<td><a href="publisher/">publisher</a></td>
<td>The publisher of the resource. For a dataset, this is the repository where it is stored.</td>
</tr>
<tr>
<td><a href="quality/">quality</a></td>
<td>A quality score for measurement.</td>
</tr>
<tr>
<td><a href="raw_value/">raw_value</a></td>
<td>The value that was specified for an annotation in its raw form; e.g. "2 cm" or "2-4 cm"</td>
</tr>
<tr>
<td><a href="relationship/">relationship</a></td>
<td>Relationship between this identifier and the entity in the <code>entity_id</code> field. If absent, it is assumed that the identifier represents the same entity in another data source.</td>
</tr>
<tr>
<td><a href="required/">required</a></td>
<td>Whether or not this parameter must be supplied.</td>
</tr>
<tr>
<td><a href="resource_description_id/">resource_description_id</a></td>
<td>Unique identifier for a description for a resource.</td>
</tr>
<tr>
<td><a href="resource_descriptions/">resource_descriptions</a></td>
<td>All resource descriptions in the schema.</td>
</tr>
<tr>
<td><a href="resource_title_id/">resource_title_id</a></td>
<td>Unique identifier for a title for a resource.</td>
</tr>
<tr>
<td><a href="resource_titles/">resource_titles</a></td>
<td>All resource titles in the schema.</td>
</tr>
<tr>
<td><a href="resource_type/">resource_type</a></td>
<td>The broad type of the source data for this object. 'dataset' is currently the only valid value supported by this schema.</td>
</tr>
<tr>
<td><a href="sample_id/">sample_id</a></td>
<td>Internal (CDM) unique identifier for a sample.<br>From the Entity table: entity_id where entity_type == 'Sample'.</td>
</tr>
<tr>
<td><a href="samples/">samples</a></td>
<td>All samples in the schema.</td>
</tr>
<tr>
<td><a href="scaffold_bp/">scaffold_bp</a></td>
<td>Total size in bp of all scaffolds</td>
</tr>
<tr>
<td><a href="scaffold_l50/">scaffold_l50</a></td>
<td>Given a set of scaffolds, the L50 is defined as the sequence length of the shortest scaffold at 50% of the total contig collection length</td>
</tr>
<tr>
<td><a href="scaffold_l90/">scaffold_l90</a></td>
<td>The L90 statistic is less than or equal to the L50 statistic; it is the length for which the collection of all scaffolds of that length or longer contains at least 90% of the sum of the lengths of all scaffolds.</td>
</tr>
<tr>
<td><a href="scaffold_logsum/">scaffold_logsum</a></td>
<td>The sum of the (length*log(length)) of all scaffolds, times some constant. Increase the contiguity, the score will increase</td>
</tr>
<tr>
<td><a href="scaffold_maximum_length/">scaffold_maximum_length</a></td>
<td>Maximum scaffold length</td>
</tr>
<tr>
<td><a href="scaffold_n50/">scaffold_n50</a></td>
<td>Given a set of scaffolds, each with its own length, the N50 count is defined as the smallest number of scaffolds whose length sum makes up half of contig collection size</td>
</tr>
<tr>
<td><a href="scaffold_n90/">scaffold_n90</a></td>
<td>Given a set of scaffolds, each with its own length, the N90 count is defined as the smallest number of scaffolds whose length sum makes up 90% of contig collection size</td>
</tr>
<tr>
<td><a href="scaffold_powersum/">scaffold_powersum</a></td>
<td>Powersum of all scaffolds is the same as logsum except that it uses the sum of (length*(length^P)) for some power P (default P=0.25).</td>
</tr>
<tr>
<td><a href="scaffolds_n_over_50K/">scaffolds_n_over_50K</a></td>
<td>The number of scaffolds longer than 50,000 base pairs.</td>
</tr>
<tr>
<td><a href="scaffolds_percent_over_50K/">scaffolds_percent_over_50K</a></td>
<td>The percentage of the total assembly length represented by scaffolds longer than 50,000 base pairs</td>
</tr>
<tr>
<td><a href="scaffolds_total_length_over_50k/">scaffolds_total_length_over_50k</a></td>
<td>The total length of scaffolds longer than 50,000 base pairs</td>
</tr>
<tr>
<td><a href="score/">score</a></td>
<td>Output from the clustering protocol indicating how closely a member matches the representative.</td>
</tr>
<tr>
<td><a href="sequence/">sequence</a></td>
<td>The protein amino acid sequence.</td>
</tr>
<tr>
<td><a href="sequence_id/">sequence_id</a></td>
<td>Internal (CDM) unique identifier for a sequence.<br>From the Entity table: entity_id where entity_type == 'Sequence'.</td>
</tr>
<tr>
<td><a href="sequences/">sequences</a></td>
<td>All sequences in the schema.</td>
</tr>
<tr>
<td><a href="source/">source</a></td>
<td>The source for a specific piece of information; should be a CDM internal ID of a source in the DataSource table.</td>
</tr>
<tr>
<td><a href="source_database/">source_database</a></td>
<td>ID of the data source from which this entity came.</td>
</tr>
<tr>
<td><a href="specific_ecosystem/">specific_ecosystem</a></td>
<td>JGI GOLD descriptor representing the most specific level of ecosystem categorization. May be "Unclassified".</td>
</tr>
<tr>
<td><a href="start/">start</a></td>
<td>The start and end coordinates of the feature are given in positive 1-based int coordinates, relative to the landmark given in column one. Start is always less than or equal to end. For features that cross the origin of a circular feature (e.g. most bacterial genomes, plasmids, and some viral genomes), the requirement for start to be less than or equal to end is satisfied by making end = the position of the end + the length of the landmark feature. For zero-length features, such as insertion sites, start equals end and the implied site is to the right of the indicated base in the direction of the landmark.</td>
</tr>
<tr>
<td><a href="statements/">statements</a></td>
<td>All statements in the schema.</td>
</tr>
<tr>
<td><a href="step/">step</a></td>
<td>Text description of a step in a protocol.</td>
</tr>
<tr>
<td><a href="step_index/">step_index</a></td>
<td>The number of the step in an ordered progression.</td>
</tr>
<tr>
<td><a href="strand/">strand</a></td>
<td>The strand of the feature.</td>
</tr>
<tr>
<td><a href="subject/">subject</a></td>
<td>The subject of the statement</td>
</tr>
<tr>
<td><a href="text_value/">text_value</a></td>
<td>The value, as a text string. This field should only be used if the value is not represented in a controlled vocabulary, ontology, or enumeration.</td>
</tr>
<tr>
<td><a href="title/">title</a></td>
<td>A string used as a title for a resource.</td>
</tr>
<tr>
<td><a href="title_type/">title_type</a></td>
<td>A descriptor for the title for cases where the contents of the <code>title</code> field is not the primary name or title.</td>
</tr>
<tr>
<td><a href="type/">type</a></td>
<td>The type of value being represented - e.g. QuantityValue, TextValue, DateTimeValue, ControlledVocabularyTermValue, etc.</td>
</tr>
<tr>
<td><a href="unit/">unit</a></td>
<td>The units used to measure the value of the variable, if applicable. Units should be expressed using the Unit Ontology or a term from UCUM.</td>
</tr>
<tr>
<td><a href="unit_cv_id/">unit_cv_id</a></td>
<td>The unit of the quantity, expressed as a CURIE from the Unit Ontology.</td>
</tr>
<tr>
<td><a href="unit_cv_label/">unit_cv_label</a></td>
<td>The unit of a quantity, expressed as the term name of a term from the Unit Ontology or UCUM.</td>
</tr>
<tr>
<td><a href="unit_string/">unit_string</a></td>
<td>Links a QuantityValue to a unit. Units should be taken from the UCUM unit collection or the Unit Ontology. This field should only be used if the unit is not present in one of those sources.</td>
</tr>
<tr>
<td><a href="updated/">updated</a></td>
<td>Date/timestamp for when the entity was updated in the CDM.</td>
</tr>
<tr>
<td><a href="url/">url</a></td>
<td>The URL from which the data was loaded.</td>
</tr>
<tr>
<td><a href="value/">value</a></td>
<td>Note the range of this slot is always a string. Only used if the triple represents a literal assertion</td>
</tr>
<tr>
<td><a href="value_cv_id/">value_cv_id</a></td>
<td>For values that are in a controlled vocabulary (CV), this attribute should capture the controlled vocabulary ID for the value.</td>
</tr>
<tr>
<td><a href="value_cv_label/">value_cv_label</a></td>
<td>For values that are in a controlled vocabulary, ontology, or enumeration, this attribute should capture the term from the controlled vocabulary.</td>
</tr>
<tr>
<td><a href="value_type/">value_type</a></td>
<td>The type(s) of the value.</td>
</tr>
<tr>
<td><a href="variable_id/">variable_id</a></td>
<td>Internal CDM unique identifier for a variable.</td>
</tr>
<tr>
<td><a href="variable_value_id/">variable_value_id</a></td>
<td>Internal CDM unique identifier for a variable value.</td>
</tr>
<tr>
<td><a href="variable_values/">variable_values</a></td>
<td>All variable values in the schema.</td>
</tr>
<tr>
<td><a href="variables/">variables</a></td>
<td>All variables in the schema.</td>
</tr>
<tr>
<td><a href="version/">version</a></td>
<td>The version of the resource. This must be an absolute version, not a relative version like 'latest'.</td>
</tr>
</tbody>
</table>
<h2 id="enumerations">Enumerations</h2>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="AttributeValueType/">AttributeValueType</a></td>
<td></td>
</tr>
<tr>
<td><a href="CdsPhaseType/">CdsPhaseType</a></td>
<td>For features of type CDS (coding sequence), the phase indicates where the feature begins with reference to the reading frame. The phase is one of the integers 0, 1, or 2, indicating the number of bases that should be removed from the beginning of this feature to reach the first base of the next codon.</td>
</tr>
<tr>
<td><a href="ClusterType/">ClusterType</a></td>
<td>The type of the entities in a cluster. Must be represented by a table in the CDM schema.</td>
</tr>
<tr>
<td><a href="ContigCollectionType/">ContigCollectionType</a></td>
<td>The type of the contig set; the type of the 'omics data set. Terms are taken from the Genomics Standards Consortium where possible. See the GSC checklists at  https://genomicsstandardsconsortium.github.io/mixs/ for the controlled vocabularies used.</td>
</tr>
<tr>
<td><a href="ContributorRole/">ContributorRole</a></td>
<td>The type of contribution made by a contributor.</td>
</tr>
<tr>
<td><a href="ContributorType/">ContributorType</a></td>
<td>The type of contributor being represented.</td>
</tr>
<tr>
<td><a href="DescriptionType/">DescriptionType</a></td>
<td>The type of text being represented.</td>
</tr>
<tr>
<td><a href="EntityType/">EntityType</a></td>
<td>The type of an entity. Must be represented by a table in the CDM schema.</td>
</tr>
<tr>
<td><a href="EventType/">EventType</a></td>
<td>The type of date being represented.</td>
</tr>
<tr>
<td><a href="ProteinEvidenceForExistence/">ProteinEvidenceForExistence</a></td>
<td>The evidence for the existence of a biological entity. See https://www.uniprot.org/help/protein_existence and https://www.ncbi.nlm.nih.gov/genbank/evidence/.</td>
</tr>
<tr>
<td><a href="ProtocolParameterType/">ProtocolParameterType</a></td>
<td>An input, an operation parameter or switch, or an output for a protocol.</td>
</tr>
<tr>
<td><a href="RefSeqStatusType/">RefSeqStatusType</a></td>
<td>RefSeq status codes, taken from https://www.ncbi.nlm.nih.gov/genbank/evidence/.</td>
</tr>
<tr>
<td><a href="RelationshipType/">RelationshipType</a></td>
<td>The relationship between two entities. For example, when a PermanentID class is used to represent objects in the CreditMetadata field <code>related_identifiers</code>, the <code>relationship_type</code> field captures the relationship between the resource being registered (A) and this ID (B).</td>
</tr>
<tr>
<td><a href="ResourceType/">ResourceType</a></td>
<td>The type of resource being represented.</td>
</tr>
<tr>
<td><a href="SequenceType/">SequenceType</a></td>
<td>The type of sequence being represented.</td>
</tr>
<tr>
<td><a href="StrandType/">StrandType</a></td>
<td>The strand that a feature appears on relative to a landmark. Also encompasses unknown or irrelevant strandedness.</td>
</tr>
<tr>
<td><a href="TitleType/">TitleType</a></td>
<td>The type of title being represented.</td>
</tr>
<tr>
<td><a href="VariableType/">VariableType</a></td>
<td>The type of the value of a variable. Should be a LinkML data type or one of the defined CDM data types.</td>
</tr>
</tbody>
</table>
<h2 id="types">Types</h2>
<table>
<thead>
<tr>
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Boolean/">Boolean</a></td>
<td>A binary (true or false) value</td>
</tr>
<tr>
<td><a href="CdmAssociationId/">CdmAssociationId</a></td>
<td>A CDM ID (cdm_id) that identifies an association in the CDM.</td>
</tr>
<tr>
<td><a href="CdmClusterId/">CdmClusterId</a></td>
<td>A CDM ID (cdm_id) that identifies a cluster in the CDM.</td>
</tr>
<tr>
<td><a href="CdmContigCollectionId/">CdmContigCollectionId</a></td>
<td>A CDM ID (cdm_id) that identifies a contig collection in the CDM.</td>
</tr>
<tr>
<td><a href="CdmContigId/">CdmContigId</a></td>
<td>A CDM ID (cdm_id) that identifies a contig in the CDM.</td>
</tr>
<tr>
<td><a href="CdmContributorId/">CdmContributorId</a></td>
<td>A CDM ID (cdm_id) that identifies a contributor in the CDM.</td>
</tr>
<tr>
<td><a href="CdmDataSourceId/">CdmDataSourceId</a></td>
<td>A CDM ID (cdm_id) that identifies a data source in the CDM.</td>
</tr>
<tr>
<td><a href="CdmEncodedFeatureId/">CdmEncodedFeatureId</a></td>
<td>A CDM ID (cdm_id) that identifies an encoded feature in the CDM.</td>
</tr>
<tr>
<td><a href="CdmFeatureId/">CdmFeatureId</a></td>
<td>A CDM ID (cdm_id) that identifies a feature in the CDM.</td>
</tr>
<tr>
<td><a href="CdmId/">CdmId</a></td>
<td>A universally unique ID, generated using uuid4, with the prefix "CDM:". Used internally in the CDM.</td>
</tr>
<tr>
<td><a href="CdmLicenseId/">CdmLicenseId</a></td>
<td>A CDM ID (cdm_id) that identifies a license in the CDM.</td>
</tr>
<tr>
<td><a href="CdmProteinId/">CdmProteinId</a></td>
<td>A CDM ID (cdm_id) that identifies a protein in the CDM.</td>
</tr>
<tr>
<td><a href="CdmProtocolId/">CdmProtocolId</a></td>
<td>A CDM ID (cdm_id) that identifies a protocol in the CDM.</td>
</tr>
<tr>
<td><a href="CdmSampleId/">CdmSampleId</a></td>
<td>A CDM ID (cdm_id) that identifies a sample in the CDM.</td>
</tr>
<tr>
<td><a href="CdmSequenceId/">CdmSequenceId</a></td>
<td>A CDM ID (cdm_id) that identifies a sequence in the CDM.</td>
</tr>
<tr>
<td><a href="Curie/">Curie</a></td>
<td>a compact URI</td>
</tr>
<tr>
<td><a href="Date/">Date</a></td>
<td>a date (year, month and day) in an idealized calendar</td>
</tr>
<tr>
<td><a href="DateOrDatetime/">DateOrDatetime</a></td>
<td>Either a date or a datetime</td>
</tr>
<tr>
<td><a href="Datetime/">Datetime</a></td>
<td>The combination of a date and time</td>
</tr>
<tr>
<td><a href="Decimal/">Decimal</a></td>
<td>A real number with arbitrary precision that conforms to the xsd:decimal specification</td>
</tr>
<tr>
<td><a href="DecimalDegree/">DecimalDegree</a></td>
<td>A decimal degree expresses latitude or longitude as decimal fractions.</td>
</tr>
<tr>
<td><a href="Double/">Double</a></td>
<td>A real number that conforms to the xsd:double specification</td>
</tr>
<tr>
<td><a href="Float/">Float</a></td>
<td>A real number that conforms to the xsd:float specification</td>
</tr>
<tr>
<td><a href="Integer/">Integer</a></td>
<td>An integer</td>
</tr>
<tr>
<td><a href="Iso8601/">Iso8601</a></td>
<td>A date in ISO 8601 format, e.g. 2024-04-05T12:34:56Z. "Z" indicates UTC time.</td>
</tr>
<tr>
<td><a href="Jsonpath/">Jsonpath</a></td>
<td>A string encoding a JSON Path. The value of the string MUST conform to JSON Point syntax and SHOULD dereference to zero or more valid objects within the current instance document when encoded in tree form.</td>
</tr>
<tr>
<td><a href="Jsonpointer/">Jsonpointer</a></td>
<td>A string encoding a JSON Pointer. The value of the string MUST conform to JSON Point syntax and SHOULD dereference to a valid object within the current instance document when encoded in tree form.</td>
</tr>
<tr>
<td><a href="LiteralAsStringType/">LiteralAsStringType</a></td>
<td>A literal represented as a string.</td>
</tr>
<tr>
<td><a href="LocalCurie/">LocalCurie</a></td>
<td>A CURIE that exists as a subject in the <code>statements</code> table (i.e. <code>statements.subject</code>). Should not be used for external identifiers.</td>
</tr>
<tr>
<td><a href="LocalCurieName/">LocalCurieName</a></td>
<td>The term name for an ontology term; should appear as an object in the <code>statements</code> table with a relationship indicating that it is the name of an ontology term.</td>
</tr>
<tr>
<td><a href="Ncname/">Ncname</a></td>
<td>Prefix part of CURIE</td>
</tr>
<tr>
<td><a href="NodeIdType/">NodeIdType</a></td>
<td>IDs are either CURIEs, IRI, or blank nodes. IRIs are wrapped in &lt;&gt;s to distinguish them from CURIEs, but in general it is good practice to populate the [prefixes][Prefixes.md] table such that they are shortened to CURIEs. Blank nodes are ids starting with <code>_:</code>.</td>
</tr>
<tr>
<td><a href="Nodeidentifier/">Nodeidentifier</a></td>
<td>A URI, CURIE or BNODE that represents a node in a model.</td>
</tr>
<tr>
<td><a href="Objectidentifier/">Objectidentifier</a></td>
<td>A URI or CURIE that represents an object in the model.</td>
</tr>
<tr>
<td><a href="Sparqlpath/">Sparqlpath</a></td>
<td>A string encoding a SPARQL Property Path. The value of the string MUST conform to SPARQL syntax and SHOULD dereference to zero or more valid objects within the current instance document when encoded as RDF.</td>
</tr>
<tr>
<td><a href="String/">String</a></td>
<td>A character string</td>
</tr>
<tr>
<td><a href="Time/">Time</a></td>
<td>A time object represents a (local) time of day, independent of any particular day</td>
</tr>
<tr>
<td><a href="Uri/">Uri</a></td>
<td>a complete URI</td>
</tr>
<tr>
<td><a href="Uriorcurie/">Uriorcurie</a></td>
<td>a URI or a CURIE</td>
</tr>
</tbody>
</table>
<h2 id="subsets">Subsets</h2>
<table>
<thead>
<tr>
<th>Subset</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td></td>
</tr>
</tbody>
</table>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": ".", "features": ["content.tabs.link"], "search": "assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="assets/javascripts/bundle.79ae519e.min.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="js/tablesort.js"></script>
      
    
  </body>
</html>