
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://kbase.github.io/cdm-schema/">
      
      
      
      
      <link rel="icon" href="assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.14">
    
    
      
        <title>cdm-schema</title>
      
    
    
      <link rel="stylesheet" href="assets/stylesheets/main.342714a4.min.css">
      
        
        <link rel="stylesheet" href="assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL(".",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#kbase_cdm" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="." title="cdm-schema" class="md-header__button md-logo" aria-label="cdm-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            cdm-schema
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Index
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/kbase/cdm-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="." title="cdm-schema" class="md-nav__button md-logo" aria-label="cdm-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    cdm-schema
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/kbase/cdm-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Index
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="." class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Index
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      Classes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      Slots
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      Enumerations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      Types
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      Subsets
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="about.md" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    About
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      Classes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      Slots
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      Enumerations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      Types
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      Subsets
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="kbase_cdm">kbase_cdm</h1>
<p>Schema for KBase CDM</p>
<p>URI: https://github.com/kbase/cdm-schema</p>
<p>Name: kbase_cdm</p>
<h2 id="classes">Classes</h2>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Any/">Any</a></td>
<td></td>
</tr>
<tr>
<td><a href="Association/">Association</a></td>
<td>An association between an object--typically an entity such as a protein or a ...</td>
</tr>
<tr>
<td><a href="AssociationXEntity/">AssociationXEntity</a></td>
<td>Links associations to entities</td>
</tr>
<tr>
<td><a href="AssociationXPublication/">AssociationXPublication</a></td>
<td>Links associations to supporting literature</td>
</tr>
<tr>
<td><a href="AttributeValue/">AttributeValue</a></td>
<td>The value for any value of a attribute for a sample</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ControlledTermValue/">ControlledTermValue</a></td>
<td>A controlled term or class from an ontology</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ControlledIdentifiedTermValue/">ControlledIdentifiedTermValue</a></td>
<td>A controlled term or class from an ontology, requiring the presence of term w...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Geolocation/">Geolocation</a></td>
<td>A normalized value for a location on the earth's surface</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="QuantityValue/">QuantityValue</a></td>
<td>A simple quantity, e</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="TextValue/">TextValue</a></td>
<td>A basic string value</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="TimestampValue/">TimestampValue</a></td>
<td>A value that is a timestamp</td>
</tr>
<tr>
<td><a href="Cluster/">Cluster</a></td>
<td>Represents an individual execution of a clustering protocol</td>
</tr>
<tr>
<td><a href="ClusterXProtein/">ClusterXProtein</a></td>
<td>Relationship representing membership of a cluster</td>
</tr>
<tr>
<td><a href="Contig/">Contig</a></td>
<td>A contig (derived from the word "contiguous") is a set of DNA segments or seq...</td>
</tr>
<tr>
<td><a href="ContigXContigCollection/">ContigXContigCollection</a></td>
<td>Captures the relationship between a contig and a contig collection; equivalen...</td>
</tr>
<tr>
<td><a href="ContigXFeature/">ContigXFeature</a></td>
<td>Captures the relationship between a contig and a feature; equivalent to featu...</td>
</tr>
<tr>
<td><a href="ContigCollection/">ContigCollection</a></td>
<td>A set of individual, overlapping contigs that represent the complete sequence...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Genome/">Genome</a></td>
<td>A contig collection with a completeness score of greater than 90% and a conta...</td>
</tr>
<tr>
<td><a href="ContigCollectionXFeature/">ContigCollectionXFeature</a></td>
<td>Captures the relationship between a contig collection and a feature; equivale...</td>
</tr>
<tr>
<td><a href="ContigCollectionXProtein/">ContigCollectionXProtein</a></td>
<td>Captures the relationship between a contig collection and a protein; equivale...</td>
</tr>
<tr>
<td><a href="Contributor/">Contributor</a></td>
<td>Represents a contributor to the resource</td>
</tr>
<tr>
<td><a href="ContributorXExperiment/">ContributorXExperiment</a></td>
<td></td>
</tr>
<tr>
<td><a href="ContributorXOrganization/">ContributorXOrganization</a></td>
<td>Captures the organization(s) to which a contributor belongs</td>
</tr>
<tr>
<td><a href="ContributorXProject/">ContributorXProject</a></td>
<td></td>
</tr>
<tr>
<td><a href="DataSource/">DataSource</a></td>
<td>The source dataset from which data within the CDM was extracted</td>
</tr>
<tr>
<td><a href="EncodedFeature/">EncodedFeature</a></td>
<td>An entity generated from a feature, such as a transcript</td>
</tr>
<tr>
<td><a href="EncodedFeatureXFeature/">EncodedFeatureXFeature</a></td>
<td>Captures the relationship between a feature and its transcription product</td>
</tr>
<tr>
<td><a href="EntailedEdge/">EntailedEdge</a></td>
<td>A relation graph edge that is inferred</td>
</tr>
<tr>
<td><a href="Entity/">Entity</a></td>
<td>A database entity</td>
</tr>
<tr>
<td><a href="Event/">Event</a></td>
<td>Something that happened</td>
</tr>
<tr>
<td><a href="Experiment/">Experiment</a></td>
<td>A discrete scientific procedure undertaken to make a discovery, test a hypoth...</td>
</tr>
<tr>
<td><a href="Feature/">Feature</a></td>
<td>A feature localized to an interval along a contig</td>
</tr>
<tr>
<td><a href="FeatureXProtein/">FeatureXProtein</a></td>
<td>Captures the relationship between a feature and a protein; equivalent to feat...</td>
</tr>
<tr>
<td><a href="FeatureAttributes/">FeatureAttributes</a></td>
<td>Additional attributes of a feature, parsed from column 9 of a GFF file</td>
</tr>
<tr>
<td><a href="GoldEnvironmentalContext/">GoldEnvironmentalContext</a></td>
<td>Environmental context, described using JGI's five level system</td>
</tr>
<tr>
<td><a href="HasHash/">HasHash</a></td>
<td></td>
</tr>
<tr>
<td><a href="HasIdentifiers/">HasIdentifiers</a></td>
<td>Adds a multivalued 'identifiers' field to an object</td>
</tr>
<tr>
<td><a href="HasNames/">HasNames</a></td>
<td>Adds a multivalued 'names' field to an object</td>
</tr>
<tr>
<td><a href="Identifier/">Identifier</a></td>
<td>An external identifier for an entity</td>
</tr>
<tr>
<td><a href="Location/">Location</a></td>
<td></td>
</tr>
<tr>
<td><a href="Measurement/">Measurement</a></td>
<td>A qualitative or quantitative observation of an attribute of an object or eve...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ProcessedMeasurement/">ProcessedMeasurement</a></td>
<td>A measurement that requires additional processing to generate a result</td>
</tr>
<tr>
<td><a href="MeasurementSet/">MeasurementSet</a></td>
<td>A series of qualitative or quantitative measurements</td>
</tr>
<tr>
<td><a href="MixsEnvironmentalContext/">MixsEnvironmentalContext</a></td>
<td>Environmental context, described using the MiXS convention of broad and local...</td>
</tr>
<tr>
<td><a href="Name/">Name</a></td>
<td>The name or label for an entity</td>
</tr>
<tr>
<td><a href="Organization/">Organization</a></td>
<td></td>
</tr>
<tr>
<td><a href="Prefix/">Prefix</a></td>
<td>Maps CURIEs to URIs</td>
</tr>
<tr>
<td><a href="Project/">Project</a></td>
<td>Administrative unit for collecting data related to a certain topic, location,...</td>
</tr>
<tr>
<td><a href="Protein/">Protein</a></td>
<td>Proteins are large, complex molecules made up of one or more long, folded cha...</td>
</tr>
<tr>
<td><a href="Protocol/">Protocol</a></td>
<td>Defined method or set of methods</td>
</tr>
<tr>
<td><a href="ProtocolParticipant/">ProtocolParticipant</a></td>
<td>Either an input or an output of a protocol</td>
</tr>
<tr>
<td><a href="Publication/">Publication</a></td>
<td>A publication (e</td>
</tr>
<tr>
<td><a href="Sample/">Sample</a></td>
<td>A material entity that can be characterised by an experiment</td>
</tr>
<tr>
<td><a href="Statements/">Statements</a></td>
<td>Represents an RDF triple</td>
</tr>
<tr>
<td><a href="Thing/">Thing</a></td>
<td>A thing in the schema</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="NamedThingWithIDs/">NamedThingWithIDs</a></td>
<td></td>
</tr>
</tbody>
</table>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Slot</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="affiliations/">affiliations</a></td>
<td>List of organizations with which the contributor is affiliated</td>
</tr>
<tr>
<td><a href="aggregator_knowledge_source/">aggregator_knowledge_source</a></td>
<td>The knowledge source that aggregated the association</td>
</tr>
<tr>
<td><a href="altitude_units/">altitude_units</a></td>
<td></td>
</tr>
<tr>
<td><a href="altitude_value/">altitude_value</a></td>
<td></td>
</tr>
<tr>
<td><a href="annotation_date/">annotation_date</a></td>
<td>The date when the annotation was made</td>
</tr>
<tr>
<td><a href="asm_score/">asm_score</a></td>
<td>A composite score for comparing contig collection quality</td>
</tr>
<tr>
<td><a href="association_id/">association_id</a></td>
<td>Unique identifier for an association</td>
</tr>
<tr>
<td><a href="attribute_name/">attribute_name</a></td>
<td>The name of the attribute</td>
</tr>
<tr>
<td><a href="attribute_value/">attribute_value</a></td>
<td>The value of the attribute</td>
</tr>
<tr>
<td><a href="base/">base</a></td>
<td>The base URI a prefix will expand to</td>
</tr>
<tr>
<td><a href="cds_phase/">cds_phase</a></td>
<td>For features of type CDS, the phase indicates where the next codon begins rel...</td>
</tr>
<tr>
<td><a href="checkm2_completeness/">checkm2_completeness</a></td>
<td>Estimate of the completeness of a contig collection (MAG or genome), estimate...</td>
</tr>
<tr>
<td><a href="checkm2_contamination/">checkm2_contamination</a></td>
<td>Estimate of the contamination of a contig collection (MAG or genome), estimat...</td>
</tr>
<tr>
<td><a href="cluster_id/">cluster_id</a></td>
<td>Internal (CDM) unique identifier</td>
</tr>
<tr>
<td><a href="comments/">comments</a></td>
<td>Any comments about the association</td>
</tr>
<tr>
<td><a href="conditions/">conditions</a></td>
<td>TBD</td>
</tr>
<tr>
<td><a href="contig_bp/">contig_bp</a></td>
<td>Total size in bp of all contigs</td>
</tr>
<tr>
<td><a href="contig_collection_id/">contig_collection_id</a></td>
<td>Internal (CDM) unique identifier for a contig collection</td>
</tr>
<tr>
<td><a href="contig_collection_type/">contig_collection_type</a></td>
<td>The type of contig collection</td>
</tr>
<tr>
<td><a href="contig_id/">contig_id</a></td>
<td>Internal (CDM) unique identifier for a contig</td>
</tr>
<tr>
<td><a href="contributor_id/">contributor_id</a></td>
<td>Internal (CDM) unique identifier for a contributor</td>
</tr>
<tr>
<td><a href="contributor_roles/">contributor_roles</a></td>
<td>List of roles played by the contributor when working on the resource</td>
</tr>
<tr>
<td><a href="contributor_type/">contributor_type</a></td>
<td>Must be either 'Person' or 'Organization'</td>
</tr>
<tr>
<td><a href="created/">created</a></td>
<td>POSIX timestamp for when the entity was created or added to the CDM</td>
</tr>
<tr>
<td><a href="created_at/">created_at</a></td>
<td>The time at which the event started or was created</td>
</tr>
<tr>
<td><a href="ctg_L50/">ctg_L50</a></td>
<td>Given a set of contigs, the L50 is defined as the sequence length of the shor...</td>
</tr>
<tr>
<td><a href="ctg_L90/">ctg_L90</a></td>
<td>The L90 statistic is less than or equal to the L50 statistic; it is the lengt...</td>
</tr>
<tr>
<td><a href="ctg_logsum/">ctg_logsum</a></td>
<td>The sum of the (length*log(length)) of all contigs, times some constant</td>
</tr>
<tr>
<td><a href="ctg_max/">ctg_max</a></td>
<td>Maximum contig length</td>
</tr>
<tr>
<td><a href="ctg_N50/">ctg_N50</a></td>
<td>Given a set of contigs, each with its own length, the N50 count is defined as...</td>
</tr>
<tr>
<td><a href="ctg_N90/">ctg_N90</a></td>
<td>Given a set of contigs, each with its own length, the N90 count is defined as...</td>
</tr>
<tr>
<td><a href="ctg_powsum/">ctg_powsum</a></td>
<td>Powersum of all contigs is the same as logsum except that it uses the sum of ...</td>
</tr>
<tr>
<td><a href="data_source/">data_source</a></td>
<td>How this entity was derived and added to the database</td>
</tr>
<tr>
<td><a href="datatype/">datatype</a></td>
<td>the rdf datatype of the value, for example, xsd:string</td>
</tr>
<tr>
<td><a href="date_accessed/">date_accessed</a></td>
<td>The date when the data was downloaded from the data source</td>
</tr>
<tr>
<td><a href="depth_units/">depth_units</a></td>
<td></td>
</tr>
<tr>
<td><a href="depth_value/">depth_value</a></td>
<td></td>
</tr>
<tr>
<td><a href="description/">description</a></td>
<td>Definition or description</td>
</tr>
<tr>
<td><a href="doi/">doi</a></td>
<td>The DOI for a protocol</td>
</tr>
<tr>
<td><a href="e_value/">e_value</a></td>
<td>The 'score' of the feature</td>
</tr>
<tr>
<td><a href="ecosystem/">ecosystem</a></td>
<td>JGI GOLD descriptor representing the top level ecosystem categorization</td>
</tr>
<tr>
<td><a href="ecosystem_category/">ecosystem_category</a></td>
<td>JGI GOLD descriptor representing the ecosystem category</td>
</tr>
<tr>
<td><a href="ecosystem_subtype/">ecosystem_subtype</a></td>
<td>JGI GOLD descriptor representing the subtype of ecosystem</td>
</tr>
<tr>
<td><a href="ecosystem_type/">ecosystem_type</a></td>
<td>JGI GOLD descriptor representing the ecosystem type</td>
</tr>
<tr>
<td><a href="elevation_units/">elevation_units</a></td>
<td></td>
</tr>
<tr>
<td><a href="elevation_value/">elevation_value</a></td>
<td></td>
</tr>
<tr>
<td><a href="encoded_by/">encoded_by</a></td>
<td>The feature(s) that encode this protein</td>
</tr>
<tr>
<td><a href="encoded_feature_id/">encoded_feature_id</a></td>
<td>Internal (CDM) unique identifier</td>
</tr>
<tr>
<td><a href="end/">end</a></td>
<td>The start and end coordinates of the feature are given in positive 1-based in...</td>
</tr>
<tr>
<td><a href="entity_id/">entity_id</a></td>
<td>Internal (CDM) unique identifier for an entity in the CDM</td>
</tr>
<tr>
<td><a href="entity_type/">entity_type</a></td>
<td>The class of the entity</td>
</tr>
<tr>
<td><a href="env_broad_scale/">env_broad_scale</a></td>
<td>Report the major environmental system the sample or specimen came from</td>
</tr>
<tr>
<td><a href="env_local_scale/">env_local_scale</a></td>
<td>Report the entity or entities which are in the sample or specimen's local vic...</td>
</tr>
<tr>
<td><a href="env_medium/">env_medium</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or sp...</td>
</tr>
<tr>
<td><a href="event_id/">event_id</a></td>
<td>Internal (CDM) unique identifier</td>
</tr>
<tr>
<td><a href="evidence_for_existence/">evidence_for_existence</a></td>
<td>The evidence that this protein exists</td>
</tr>
<tr>
<td><a href="evidence_type/">evidence_type</a></td>
<td>The type of evidence supporting the association</td>
</tr>
<tr>
<td><a href="experiment_id/">experiment_id</a></td>
<td>Internal (CDM) unique identifier for an experiment</td>
</tr>
<tr>
<td><a href="family_name/">family_name</a></td>
<td>The family name(s) of the contributor</td>
</tr>
<tr>
<td><a href="feature_id/">feature_id</a></td>
<td>Internal (CDM) unique identifier for a feature</td>
</tr>
<tr>
<td><a href="gap_pct/">gap_pct</a></td>
<td>The gap size percentage of all scaffolds</td>
</tr>
<tr>
<td><a href="gc_avg/">gc_avg</a></td>
<td>The average GC content of the contig collection, expressed as a percentage</td>
</tr>
<tr>
<td><a href="gc_content/">gc_content</a></td>
<td>GC content of the contig, expressed as a percentage</td>
</tr>
<tr>
<td><a href="gc_std/">gc_std</a></td>
<td>The standard deviation of GC content across the contig collection</td>
</tr>
<tr>
<td><a href="generated_by/">generated_by</a></td>
<td>The algorithm or procedure that generated the feature</td>
</tr>
<tr>
<td><a href="given_name/">given_name</a></td>
<td>The given name(s) of the contributor</td>
</tr>
<tr>
<td><a href="gold_environmental_context/">gold_environmental_context</a></td>
<td>The environmental context for this event</td>
</tr>
<tr>
<td><a href="has_participant/">has_participant</a></td>
<td>Participants in an experiment</td>
</tr>
<tr>
<td><a href="has_stop_codon/">has_stop_codon</a></td>
<td>Captures whether or not the sequence includes a stop coordinates</td>
</tr>
<tr>
<td><a href="hash/">hash</a></td>
<td>A hash value generated from one or more object attributes that serves to ensu...</td>
</tr>
<tr>
<td><a href="id/">id</a></td>
<td>An identifier for an element</td>
</tr>
<tr>
<td><a href="identifier/">identifier</a></td>
<td>Fully-qualifier URL or CURIE used as an identifier for an entity</td>
</tr>
<tr>
<td><a href="identifiers/">identifiers</a></td>
<td>URIs or CURIEs used to refer to this entity</td>
</tr>
<tr>
<td><a href="inputs/">inputs</a></td>
<td>The inputs for a protocol; may be software parameters, experimental reagents,...</td>
</tr>
<tr>
<td><a href="language/">language</a></td>
<td>the human language in which the value is encoded, e</td>
</tr>
<tr>
<td><a href="latitude/">latitude</a></td>
<td></td>
</tr>
<tr>
<td><a href="length/">length</a></td>
<td>Length of the contig in bp</td>
</tr>
<tr>
<td><a href="location/">location</a></td>
<td>The location for this event</td>
</tr>
<tr>
<td><a href="longitude/">longitude</a></td>
<td></td>
</tr>
<tr>
<td><a href="maximum_numeric_value/">maximum_numeric_value</a></td>
<td></td>
</tr>
<tr>
<td><a href="measurement_id/">measurement_id</a></td>
<td>Internal (CDM) unique identifier</td>
</tr>
<tr>
<td><a href="minimum_numeric_value/">minimum_numeric_value</a></td>
<td></td>
</tr>
<tr>
<td><a href="mixs_environmental_context/">mixs_environmental_context</a></td>
<td>The environmental context for this event</td>
</tr>
<tr>
<td><a href="n_contigs/">n_contigs</a></td>
<td>Total number of contigs</td>
</tr>
<tr>
<td><a href="n_scaffolds/">n_scaffolds</a></td>
<td>Total number of scaffolds</td>
</tr>
<tr>
<td><a href="name/">name</a></td>
<td>A string used as a name or title</td>
</tr>
<tr>
<td><a href="names/">names</a></td>
<td>Names, alternative names, and synonyms for an entity</td>
</tr>
<tr>
<td><a href="negated/">negated</a></td>
<td>If true, the relationship between the subject and object is negated</td>
</tr>
<tr>
<td><a href="numeric_value/">numeric_value</a></td>
<td></td>
</tr>
<tr>
<td><a href="object/">object</a></td>
<td>Note the range of this slot is always a node</td>
</tr>
<tr>
<td><a href="organization_id/">organization_id</a></td>
<td>Internal (CDM) unique identifier</td>
</tr>
<tr>
<td><a href="outputs/">outputs</a></td>
<td>The outputs of a protocol; may be physical entities, files, etc</td>
</tr>
<tr>
<td><a href="p_value/">p_value</a></td>
<td>The 'score' of the feature</td>
</tr>
<tr>
<td><a href="part_of/">part_of</a></td>
<td>The project to which this experiment belongs</td>
</tr>
<tr>
<td><a href="person_id/">person_id</a></td>
<td>Internal (CDM) unique identifier for a contributor</td>
</tr>
<tr>
<td><a href="predicate/">predicate</a></td>
<td>The predicate of the statement</td>
</tr>
<tr>
<td><a href="prefix/">prefix</a></td>
<td>A standardized prefix such as 'GO' or 'rdf' or 'FlyBase'</td>
</tr>
<tr>
<td><a href="primary_knowledge_source/">primary_knowledge_source</a></td>
<td>The knowledge source that created the association</td>
</tr>
<tr>
<td><a href="project_id/">project_id</a></td>
<td>Internal (CDM) unique identifier for a project</td>
</tr>
<tr>
<td><a href="protein_id/">protein_id</a></td>
<td>Internal (CDM) unique identifier for a protein within a cluster</td>
</tr>
<tr>
<td><a href="protocol_id/">protocol_id</a></td>
<td>Protocol used to generate the cluster</td>
</tr>
<tr>
<td><a href="protocol_participant_id/">protocol_participant_id</a></td>
<td>Internal (CDM) unique identifier</td>
</tr>
<tr>
<td><a href="publication_id/">publication_id</a></td>
<td>Unique identifier for a publication</td>
</tr>
<tr>
<td><a href="publications/">publications</a></td>
<td>Publications that support the association</td>
</tr>
<tr>
<td><a href="quality/">quality</a></td>
<td>The quality of the measurement, indicating the confidence that one can have i...</td>
</tr>
<tr>
<td><a href="raw_value/">raw_value</a></td>
<td></td>
</tr>
<tr>
<td><a href="sample_id/">sample_id</a></td>
<td>Internal (CDM) unique identifier</td>
</tr>
<tr>
<td><a href="scaf_bp/">scaf_bp</a></td>
<td>Total size in bp of all scaffolds</td>
</tr>
<tr>
<td><a href="scaf_L50/">scaf_L50</a></td>
<td>Given a set of scaffolds, the L50 is defined as the sequence length of the sh...</td>
</tr>
<tr>
<td><a href="scaf_L90/">scaf_L90</a></td>
<td>The L90 statistic is less than or equal to the L50 statistic; it is the lengt...</td>
</tr>
<tr>
<td><a href="scaf_l_gt50k/">scaf_l_gt50k</a></td>
<td>The total length of scaffolds longer than 50,000 base pairs</td>
</tr>
<tr>
<td><a href="scaf_logsum/">scaf_logsum</a></td>
<td>The sum of the (length*log(length)) of all scaffolds, times some constant</td>
</tr>
<tr>
<td><a href="scaf_max/">scaf_max</a></td>
<td>Maximum scaffold length</td>
</tr>
<tr>
<td><a href="scaf_N50/">scaf_N50</a></td>
<td>Given a set of scaffolds, each with its own length, the N50 count is defined ...</td>
</tr>
<tr>
<td><a href="scaf_N90/">scaf_N90</a></td>
<td>Given a set of scaffolds, each with its own length, the N90 count is defined ...</td>
</tr>
<tr>
<td><a href="scaf_n_gt50K/">scaf_n_gt50K</a></td>
<td>The number of scaffolds longer than 50,000 base pairs</td>
</tr>
<tr>
<td><a href="scaf_pct_gt50K/">scaf_pct_gt50K</a></td>
<td>The percentage of the total assembly length represented by scaffolds longer t...</td>
</tr>
<tr>
<td><a href="scaf_powsum/">scaf_powsum</a></td>
<td>Powersum of all scaffolds is the same as logsum except that it uses the sum o...</td>
</tr>
<tr>
<td><a href="score/">score</a></td>
<td>Output from clustering protocol</td>
</tr>
<tr>
<td><a href="sequence/">sequence</a></td>
<td>The protein amino acid sequence</td>
</tr>
<tr>
<td><a href="source/">source</a></td>
<td>The name of the data source</td>
</tr>
<tr>
<td><a href="source_database/">source_database</a></td>
<td>ID of the data source from which this entity came</td>
</tr>
<tr>
<td><a href="specific_ecosystem/">specific_ecosystem</a></td>
<td>JGI GOLD descriptor representing the most specific level of ecosystem categor...</td>
</tr>
<tr>
<td><a href="start/">start</a></td>
<td>The start and end coordinates of the feature are given in positive 1-based in...</td>
</tr>
<tr>
<td><a href="strand/">strand</a></td>
<td>The strand of the feature</td>
</tr>
<tr>
<td><a href="subject/">subject</a></td>
<td>The subject of the statement</td>
</tr>
<tr>
<td><a href="supporting_objects/">supporting_objects</a></td>
<td>Objects that support the association</td>
</tr>
<tr>
<td><a href="term/">term</a></td>
<td></td>
</tr>
<tr>
<td><a href="type/">type</a></td>
<td></td>
</tr>
<tr>
<td><a href="unit/">unit</a></td>
<td></td>
</tr>
<tr>
<td><a href="updated/">updated</a></td>
<td>POSIX timestamp for when the entity was updated in the CDM</td>
</tr>
<tr>
<td><a href="url/">url</a></td>
<td>The URL from which the data was loaded</td>
</tr>
<tr>
<td><a href="value/">value</a></td>
<td>Note the range of this slot is always a string</td>
</tr>
<tr>
<td><a href="version/">version</a></td>
<td>For versioned data sources, the version of the dataset</td>
</tr>
</tbody>
</table>
<h2 id="enumerations">Enumerations</h2>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="CdsPhaseType/">CdsPhaseType</a></td>
<td>For features of type CDS (coding sequence), the phase indicates where the fea...</td>
</tr>
<tr>
<td><a href="ContigCollectionType/">ContigCollectionType</a></td>
<td>The type of the contig set; the type of the 'omics data set</td>
</tr>
<tr>
<td><a href="ContributorRole/">ContributorRole</a></td>
<td>The role of a contributor to a resource</td>
</tr>
<tr>
<td><a href="ContributorType/">ContributorType</a></td>
<td>The type of contributor being represented</td>
</tr>
<tr>
<td><a href="EntityType/">EntityType</a></td>
<td>The type of an entity</td>
</tr>
<tr>
<td><a href="ProteinEvidenceForExistence/">ProteinEvidenceForExistence</a></td>
<td>The evidence for the existence of a biological entity</td>
</tr>
<tr>
<td><a href="RefSeqStatusType/">RefSeqStatusType</a></td>
<td>RefSeq status codes, taken from https://www</td>
</tr>
<tr>
<td><a href="StrandType/">StrandType</a></td>
<td>The strand that a feature appears on relative to a landmark</td>
</tr>
</tbody>
</table>
<h2 id="types">Types</h2>
<table>
<thead>
<tr>
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Boolean/">Boolean</a></td>
<td>A binary (true or false) value</td>
</tr>
<tr>
<td><a href="Curie/">Curie</a></td>
<td>a compact URI</td>
</tr>
<tr>
<td><a href="Date/">Date</a></td>
<td>a date (year, month and day) in an idealized calendar</td>
</tr>
<tr>
<td><a href="DateOrDatetime/">DateOrDatetime</a></td>
<td>Either a date or a datetime</td>
</tr>
<tr>
<td><a href="Datetime/">Datetime</a></td>
<td>The combination of a date and time</td>
</tr>
<tr>
<td><a href="Decimal/">Decimal</a></td>
<td>A real number with arbitrary precision that conforms to the xsd:decimal speci...</td>
</tr>
<tr>
<td><a href="Double/">Double</a></td>
<td>A real number that conforms to the xsd:double specification</td>
</tr>
<tr>
<td><a href="Float/">Float</a></td>
<td>A real number that conforms to the xsd:float specification</td>
</tr>
<tr>
<td><a href="Integer/">Integer</a></td>
<td>An integer</td>
</tr>
<tr>
<td><a href="Jsonpath/">Jsonpath</a></td>
<td>A string encoding a JSON Path</td>
</tr>
<tr>
<td><a href="Jsonpointer/">Jsonpointer</a></td>
<td>A string encoding a JSON Pointer</td>
</tr>
<tr>
<td><a href="LiteralAsStringType/">LiteralAsStringType</a></td>
<td></td>
</tr>
<tr>
<td><a href="Ncname/">Ncname</a></td>
<td>Prefix part of CURIE</td>
</tr>
<tr>
<td><a href="NodeIdType/">NodeIdType</a></td>
<td>IDs are either CURIEs, IRI, or blank nodes</td>
</tr>
<tr>
<td><a href="Nodeidentifier/">Nodeidentifier</a></td>
<td>A URI, CURIE or BNODE that represents a node in a model</td>
</tr>
<tr>
<td><a href="Objectidentifier/">Objectidentifier</a></td>
<td>A URI or CURIE that represents an object in the model</td>
</tr>
<tr>
<td><a href="Sparqlpath/">Sparqlpath</a></td>
<td>A string encoding a SPARQL Property Path</td>
</tr>
<tr>
<td><a href="String/">String</a></td>
<td>A character string</td>
</tr>
<tr>
<td><a href="Time/">Time</a></td>
<td>A time object represents a (local) time of day, independent of any particular...</td>
</tr>
<tr>
<td><a href="Uri/">Uri</a></td>
<td>a complete URI</td>
</tr>
<tr>
<td><a href="Uriorcurie/">Uriorcurie</a></td>
<td>a URI or a CURIE</td>
</tr>
<tr>
<td><a href="UUID/">UUID</a></td>
<td>A universally unique ID generated the KBase CDM namespace, generating using u...</td>
</tr>
</tbody>
</table>
<h2 id="subsets">Subsets</h2>
<table>
<thead>
<tr>
<th>Subset</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td></td>
</tr>
</tbody>
</table>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": ".", "features": ["content.tabs.link"], "search": "assets/javascripts/workers/search.d50fe291.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="assets/javascripts/bundle.13a4f30d.min.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="javascripts/tablesort.js"></script>
      
    
  </body>
</html>