
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://kbase.github.io/cdm-schema/">
      
      
      
        <link rel="next" href="pages/about/">
      
      
      <link rel="icon" href="assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.19">
    
    
      
        <title>cdm-schema</title>
      
    
    
      <link rel="stylesheet" href="assets/stylesheets/main.7e37652d.min.css">
      
        
        <link rel="stylesheet" href="assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL(".",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#cdm_schema" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="." title="cdm-schema" class="md-header__button md-logo" aria-label="cdm-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            cdm-schema
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Index
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/kbase/cdm-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="." title="cdm-schema" class="md-nav__button md-logo" aria-label="cdm-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    cdm-schema
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/kbase/cdm-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Index
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="." class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Index
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      Classes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      Slots
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      Enumerations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      Types
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      Subsets
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="pages/about/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    About
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      Classes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      Slots
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      Enumerations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      Types
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      Subsets
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="cdm_schema">cdm_schema</h1>
<p>Schema for KBase CDM</p>
<p>URI: http://kbase.github.io/cdm-schema/cdm_schema</p>
<p>Name: cdm_schema</p>
<h2 id="classes">Classes</h2>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Any/">Any</a></td>
<td>Used as a range for slots that have more than one possible type.</td>
</tr>
<tr>
<td><a href="Table/">Table</a></td>
<td>root class for all schema entities</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Association/">Association</a></td>
<td>An association between an object--typically an entity such as a protein or a feature--and a classification system or ontology, such as the Gene Ontology, the Enzyme Classification, or TIGRFAMS domains.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="AssociationXPublication/">AssociationXPublication</a></td>
<td>Links associations to supporting literature.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="AssociationXSupportingObject/">AssociationXSupportingObject</a></td>
<td>Links associations to entities to capture supporting objects in an association. ay be a biological entity, such as a protein or feature, or a URL to a resource (e.g. a publication) that supports the association. Where possible, CDM identifiers should be used.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="AttributeValue/">AttributeValue</a></td>
<td>A generic class for capturing tag-value information in a structured form.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Geolocation/">Geolocation</a></td>
<td>A normalized value for a location on the earth's surface</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="QuantityValue/">QuantityValue</a></td>
<td>A simple quantity, e.g. 2cm. May be used to describe a range using the minimum_value and maximum_value fields.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Measurement/">Measurement</a></td>
<td>A qualitative or quantitative observation of an attribute of an object or event against a standardized scale, to enable it to be compared with other objects or events.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ProcessedMeasurement/">ProcessedMeasurement</a></td>
<td>A measurement that requires additional processing to generate a result.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="TextValue/">TextValue</a></td>
<td>A basic string value</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="AttributeValueEntity/">AttributeValueEntity</a></td>
<td>Represents the link between an entity and its attribute values.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Cluster/">Cluster</a></td>
<td>Represents an individual execution of a clustering protocol. See the ClusterMember class for clustering results.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ClusterMember/">ClusterMember</a></td>
<td>Relationship representing membership of a cluster. An optional score can be assigned to each cluster member.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Contig/">Contig</a></td>
<td>A contig (derived from the word "contiguous") is a set of DNA segments or sequences that overlap in a way that provides a contiguous representation of a genomic region. A contig should not contain any gaps.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigXContigCollection/">ContigXContigCollection</a></td>
<td>Captures the relationship between a contig and a contig collection; equivalent to contig part-of contig collection.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigXEncodedFeature/">ContigXEncodedFeature</a></td>
<td>Captures the relationship between a contig and an encoded feature.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigXFeature/">ContigXFeature</a></td>
<td>Captures the relationship between a contig and a feature; equivalent to feature part-of contig.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigXProtein/">ContigXProtein</a></td>
<td>Captures the relationship between a contig and a protein; equivalent to protein is ribosomal translation of (http://purl.obolibrary.org/obo/RO_0002512) contig.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigCollection/">ContigCollection</a></td>
<td>A set of individual, overlapping contigs that represent the complete sequenced genome of an organism.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigCollectionXEncodedFeature/">ContigCollectionXEncodedFeature</a></td>
<td>Captures the relationship between a contig collection and an encoded feature.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigCollectionXFeature/">ContigCollectionXFeature</a></td>
<td>Captures the relationship between a contig collection and a feature; equivalent to feature part-of contig collection.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContigCollectionXProtein/">ContigCollectionXProtein</a></td>
<td>Captures the relationship between a contig collection and a protein; equivalent to protein is ribosomal translation of (http://purl.obolibrary.org/obo/RO_0002512) contig collection.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Contributor/">Contributor</a></td>
<td>Represents a contributor to the resource.<br><br>Contributors must have a 'contributor_type', either 'Person' or 'Organization', and<br>one of the 'name' fields: either 'given_name' and 'family_name' (for a person), or 'name' (for an organization or a person).<br><br>The 'contributor_role' field takes values from the DataCite and CRediT contributor<br>roles vocabularies. For more information on these resources and choosing<br>appropriate roles, please see the following links:<br><br>DataCite contributor roles: https://support.datacite.org/docs/datacite-metadata-schema-v44-recommended-and-optional-properties#7a-contributortype<br><br>CRediT contributor role taxonomy: https://credit.niso.org</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContributorXRoleXExperiment/">ContributorXRoleXExperiment</a></td>
<td></td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ContributorXRoleXProject/">ContributorXRoleXProject</a></td>
<td></td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="DataSource/">DataSource</a></td>
<td>The source dataset from which data within the CDM was extracted. This might be an API query; a set of files downloaded from a website or uploaded by a user; a database dump; etc. A given data source should have either version information (e.g. UniProt's release number) or an access date to allow the original raw data dump to be recapitulated.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="EncodedFeature/">EncodedFeature</a></td>
<td>An entity generated from a feature, such as a transcript.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="EncodedFeatureXFeature/">EncodedFeatureXFeature</a></td>
<td>Captures the relationship between a feature and its transcription product.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="EntailedEdge/">EntailedEdge</a></td>
<td>A relation graph edge that is inferred</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Entity/">Entity</a></td>
<td>A database entity.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="EntityXMeasurement/">EntityXMeasurement</a></td>
<td>Captures a measurement made on an entity.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Event/">Event</a></td>
<td>Something that happened.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Experiment/">Experiment</a></td>
<td>A discrete scientific procedure undertaken to make a discovery, test a hypothesis, or demonstrate a known fact.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ExperimentXProject/">ExperimentXProject</a></td>
<td>Captures the relationship between an experiment and the project that it is a part of.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ExperimentXSample/">ExperimentXSample</a></td>
<td>Represents the participation of a sample in an experiment.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Feature/">Feature</a></td>
<td>A feature localized to an interval along a contig.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="FeatureXProtein/">FeatureXProtein</a></td>
<td>Captures the relationship between a feature and a protein; equivalent to feature encodes protein.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="GoldEnvironmentalContext/">GoldEnvironmentalContext</a></td>
<td>Environmental context, described using JGI's five level system.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="IdentifiedEntity/">IdentifiedEntity</a></td>
<td>Represents the link between an entity and its identifiers.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Identifier/">Identifier</a></td>
<td>A string used as a resolvable (external) identifier for an entity. This should be a URI or CURIE. If the string cannot be resolved to an URL, it should be added as a 'name' instead.<br><br>This table is used for capturing external IDs. The internal CDM identifier should be used in the *_id field (e.g. feature_id, protein_id, contig_collection_id).</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="MixsEnvironmentalContext/">MixsEnvironmentalContext</a></td>
<td>Environmental context, described using the MiXS convention of broad and local environment, plus the medium.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Name/">Name</a></td>
<td>A string used as the name or label for an entity. This may be a primary name, alternative name, synonym, acronym, or any other label used to refer to an entity.<br><br>Identifiers that look like CURIEs or database references, but which cannot be resolved using bioregistry or identifiers.org should be added as names.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="NamedEntity/">NamedEntity</a></td>
<td>Represents the link between an entity and its names.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Prefix/">Prefix</a></td>
<td>Maps CURIEs to URIs</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Project/">Project</a></td>
<td>Administrative unit for collecting data related to a certain topic, location, data type, grant funding, and so on.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Protein/">Protein</a></td>
<td>Proteins are large, complex molecules made up of one or more long, folded chains of amino acids, whose sequences are determined by the DNA sequence of the protein-encoding gene.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Protocol/">Protocol</a></td>
<td>Defined method or set of methods.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ProtocolXProtocolParticipant/">ProtocolXProtocolParticipant</a></td>
<td></td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ProtocolParticipant/">ProtocolParticipant</a></td>
<td>Either an input or an output of a protocol.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Publication/">Publication</a></td>
<td>A publication (e.g. journal article).</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Sample/">Sample</a></td>
<td>A material entity that can be characterised by an experiment.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Sequence/">Sequence</a></td>
<td></td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Statements/">Statements</a></td>
<td>Represents an RDF triple</td>
</tr>
</tbody>
</table>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Slot</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="aggregator_knowledge_source/">aggregator_knowledge_source</a></td>
<td>The knowledge source that aggregated the association. Should be a UUID from the DataSource table.</td>
</tr>
<tr>
<td><a href="annotation_date/">annotation_date</a></td>
<td>The date when the annotation was made.</td>
</tr>
<tr>
<td><a href="asm_score/">asm_score</a></td>
<td>A composite score for comparing contig collection quality</td>
</tr>
<tr>
<td><a href="association_id/">association_id</a></td>
<td>Internal (CDM) unique identifier for an association.</td>
</tr>
<tr>
<td><a href="attribute_cv_term_id/">attribute_cv_term_id</a></td>
<td>If the attribute is a term from a controlled vocabulary, the ID of the term.</td>
</tr>
<tr>
<td><a href="attribute_name/">attribute_name</a></td>
<td>The attribute being captured in this annotation.</td>
</tr>
<tr>
<td><a href="base/">base</a></td>
<td>The base URI a prefix will expand to</td>
</tr>
<tr>
<td><a href="cds_phase/">cds_phase</a></td>
<td>For features of type CDS, the phase indicates where the next codon begins relative to the 5' end (where the 5' end of the CDS is relative to the strand of the CDS feature) of the current CDS feature. cds_phase is required if the feature type is CDS.</td>
</tr>
<tr>
<td><a href="checkm2_completeness/">checkm2_completeness</a></td>
<td>Estimate of the completeness of a contig collection (MAG or genome), estimated by CheckM2 tool</td>
</tr>
<tr>
<td><a href="checkm2_contamination/">checkm2_contamination</a></td>
<td>Estimate of the contamination of a contig collection (MAG or genome), estimated by CheckM2 tool</td>
</tr>
<tr>
<td><a href="checksum/">checksum</a></td>
<td>The checksum of the sequence, used to verify its integrity.</td>
</tr>
<tr>
<td><a href="cluster_id/">cluster_id</a></td>
<td>Internal (CDM) unique identifier for a cluster.</td>
</tr>
<tr>
<td><a href="comments/">comments</a></td>
<td>Any comments about the association.</td>
</tr>
<tr>
<td><a href="contig_bp/">contig_bp</a></td>
<td>Total size in bp of all contigs</td>
</tr>
<tr>
<td><a href="contig_collection_id/">contig_collection_id</a></td>
<td>Internal (CDM) unique identifier for a contig collection.</td>
</tr>
<tr>
<td><a href="contig_collection_type/">contig_collection_type</a></td>
<td>The type of contig collection.</td>
</tr>
<tr>
<td><a href="contig_id/">contig_id</a></td>
<td>Internal (CDM) unique identifier for a contig.</td>
</tr>
<tr>
<td><a href="contributor_id/">contributor_id</a></td>
<td>Internal (CDM) unique identifier for a contributor.</td>
</tr>
<tr>
<td><a href="contributor_role/">contributor_role</a></td>
<td>Role(s) played by the contributor when working on the experiment. If more than one role was played, additional rows should be added to represent each role.</td>
</tr>
<tr>
<td><a href="contributor_type/">contributor_type</a></td>
<td>Must be either 'Person' or 'Organization'.</td>
</tr>
<tr>
<td><a href="created/">created</a></td>
<td>Date/timestamp for when the entity was created or added to the CDM.</td>
</tr>
<tr>
<td><a href="created_at/">created_at</a></td>
<td>The time at which the event started or was created.</td>
</tr>
<tr>
<td><a href="ctg_L50/">ctg_L50</a></td>
<td>Given a set of contigs, the L50 is defined as the sequence length of the shortest contig at 50% of the total contig collection length</td>
</tr>
<tr>
<td><a href="ctg_L90/">ctg_L90</a></td>
<td>The L90 statistic is less than or equal to the L50 statistic; it is the length for which the collection of all contigs of that length or longer contains at least 90% of the sum of the lengths of all contigs</td>
</tr>
<tr>
<td><a href="ctg_logsum/">ctg_logsum</a></td>
<td>The sum of the (length*log(length)) of all contigs, times some constant.</td>
</tr>
<tr>
<td><a href="ctg_max/">ctg_max</a></td>
<td>Maximum contig length</td>
</tr>
<tr>
<td><a href="ctg_N50/">ctg_N50</a></td>
<td>Given a set of contigs, each with its own length, the N50 count is defined as the smallest number_of_contigs whose length sum makes up half of contig collection size</td>
</tr>
<tr>
<td><a href="ctg_N90/">ctg_N90</a></td>
<td>Given a set of contigs, each with its own length, the N90 count is defined as the smallest number of contigs whose length sum makes up 90% of contig collection size</td>
</tr>
<tr>
<td><a href="ctg_powsum/">ctg_powsum</a></td>
<td>Powersum of all contigs is the same as logsum except that it uses the sum of (length*(length^P)) for some power P (default P=0.25)</td>
</tr>
<tr>
<td><a href="data_source_created/">data_source_created</a></td>
<td>Date/timestamp for when the entity was created or added to the data source.</td>
</tr>
<tr>
<td><a href="data_source_entity_id/">data_source_entity_id</a></td>
<td>The primary ID of the entity at the data source.</td>
</tr>
<tr>
<td><a href="data_source_id/">data_source_id</a></td>
<td>Internal (CDM) unique identifier for a data source.</td>
</tr>
<tr>
<td><a href="data_source_updated/">data_source_updated</a></td>
<td>Date/timestamp for when the entity was updated in the data source.</td>
</tr>
<tr>
<td><a href="datatype/">datatype</a></td>
<td>the rdf datatype of the value, for example, xsd:string</td>
</tr>
<tr>
<td><a href="date_accessed/">date_accessed</a></td>
<td>The date when the data was downloaded from the data source.</td>
</tr>
<tr>
<td><a href="description/">description</a></td>
<td>Brief textual definition or description.</td>
</tr>
<tr>
<td><a href="doi/">doi</a></td>
<td>The DOI for a protocol.</td>
</tr>
<tr>
<td><a href="e_value/">e_value</a></td>
<td>The 'score' of the feature. The semantics of this field are ill-defined. E-values should be used for sequence similarity features.</td>
</tr>
<tr>
<td><a href="ecosystem/">ecosystem</a></td>
<td>JGI GOLD descriptor representing the top level ecosystem categorization.</td>
</tr>
<tr>
<td><a href="ecosystem_category/">ecosystem_category</a></td>
<td>JGI GOLD descriptor representing the ecosystem category.</td>
</tr>
<tr>
<td><a href="ecosystem_subtype/">ecosystem_subtype</a></td>
<td>JGI GOLD descriptor representing the subtype of ecosystem. May be "Unclassified".</td>
</tr>
<tr>
<td><a href="ecosystem_type/">ecosystem_type</a></td>
<td>JGI GOLD descriptor representing the ecosystem type. May be "Unclassified".</td>
</tr>
<tr>
<td><a href="encoded_feature_id/">encoded_feature_id</a></td>
<td>Internal (CDM) unique identifier for an encoded feature.</td>
</tr>
<tr>
<td><a href="end/">end</a></td>
<td>The start and end coordinates of the feature are given in positive 1-based int coordinates, relative to the landmark given in column one. Start is always less than or equal to end. For features that cross the origin of a circular feature (e.g. most bacterial genomes, plasmids, and some viral genomes), the requirement for start to be less than or equal to end is satisfied by making end = the position of the end + the length of the landmark feature. For zero-length features, such as insertion sites, start equals end and the implied site is to the right of the indicated base in the direction of the landmark.</td>
</tr>
<tr>
<td><a href="entity_id/">entity_id</a></td>
<td>Internal (CDM) unique identifier for an entity.</td>
</tr>
<tr>
<td><a href="entity_type/">entity_type</a></td>
<td>Type of entity being clustered.</td>
</tr>
<tr>
<td><a href="env_broad_scale/">env_broad_scale</a></td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO's biome class: http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
</tr>
<tr>
<td><a href="env_local_scale/">env_local_scale</a></td>
<td>Report the entity or entities which are in the sample or specimen's local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
</tr>
<tr>
<td><a href="env_medium/">env_medium</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
</tr>
<tr>
<td><a href="event_id/">event_id</a></td>
<td>Internal (CDM) unique identifier for an event.</td>
</tr>
<tr>
<td><a href="evidence_for_existence/">evidence_for_existence</a></td>
<td>The evidence that this protein exists. For example, the protein may have been isolated from a cell, or it may be predicted based on sequence features.</td>
</tr>
<tr>
<td><a href="evidence_type/">evidence_type</a></td>
<td>The type of evidence supporting the association. Should be a term from the Evidence and Conclusion Ontology (ECO).</td>
</tr>
<tr>
<td><a href="experiment_id/">experiment_id</a></td>
<td>Internal (CDM) unique identifier for an experiment.</td>
</tr>
<tr>
<td><a href="family_name/">family_name</a></td>
<td>The family name(s) of the contributor.</td>
</tr>
<tr>
<td><a href="feature_id/">feature_id</a></td>
<td>Internal (CDM) unique identifier for a feature.</td>
</tr>
<tr>
<td><a href="gap_pct/">gap_pct</a></td>
<td>The gap size percentage of all scaffolds</td>
</tr>
<tr>
<td><a href="gc_avg/">gc_avg</a></td>
<td>The average GC content of the contig collection, expressed as a percentage</td>
</tr>
<tr>
<td><a href="gc_content/">gc_content</a></td>
<td>GC content of the contig, expressed as a percentage.</td>
</tr>
<tr>
<td><a href="gc_std/">gc_std</a></td>
<td>The standard deviation of GC content across the contig collection</td>
</tr>
<tr>
<td><a href="given_name/">given_name</a></td>
<td>The given name(s) of the contributor.</td>
</tr>
<tr>
<td><a href="gold_environmental_context_id/">gold_environmental_context_id</a></td>
<td>Internal (CDM) unique identifier for a GOLD environmental context.</td>
</tr>
<tr>
<td><a href="has_stop_codon/">has_stop_codon</a></td>
<td>Captures whether or not the sequence includes stop coordinates.</td>
</tr>
<tr>
<td><a href="hash/">hash</a></td>
<td>A hash value generated from one or more object attributes that serves to ensure the entity is unique.</td>
</tr>
<tr>
<td><a href="id/">id</a></td>
<td>An identifier for an element. Note blank node ids are not unique across databases</td>
</tr>
<tr>
<td><a href="identifier/">identifier</a></td>
<td>Fully-qualified URL or CURIE used as an identifier for an entity.</td>
</tr>
<tr>
<td><a href="is_representative/">is_representative</a></td>
<td>Whether or not this member is the representative for the cluster. If 'is_representative' is false, it is assumed that this is a cluster member.</td>
</tr>
<tr>
<td><a href="is_seed/">is_seed</a></td>
<td>Whether or not this is the seed for this cluster.</td>
</tr>
<tr>
<td><a href="language/">language</a></td>
<td>the human language in which the value is encoded, e.g. 'en'</td>
</tr>
<tr>
<td><a href="latitude/">latitude</a></td>
<td></td>
</tr>
<tr>
<td><a href="length/">length</a></td>
<td>Length of the contig in bp.</td>
</tr>
<tr>
<td><a href="location/">location</a></td>
<td>The location for this event. May be described in terms of coordinates.</td>
</tr>
<tr>
<td><a href="longitude/">longitude</a></td>
<td></td>
</tr>
<tr>
<td><a href="maximum_value/">maximum_value</a></td>
<td>If the quantity describes a range, represents the upper bound of the range.</td>
</tr>
<tr>
<td><a href="measurement_id/">measurement_id</a></td>
<td>Internal (CDM) unique identifier for a measurement.</td>
</tr>
<tr>
<td><a href="minimum_value/">minimum_value</a></td>
<td>If the quantity describes a range, represents the lower bound of the range.</td>
</tr>
<tr>
<td><a href="mixs_environmental_context_id/">mixs_environmental_context_id</a></td>
<td>Internal (CDM) unique identifier for a mixs environmental context.</td>
</tr>
<tr>
<td><a href="n_contigs/">n_contigs</a></td>
<td>Total number of contigs</td>
</tr>
<tr>
<td><a href="n_scaffolds/">n_scaffolds</a></td>
<td>Total number of scaffolds</td>
</tr>
<tr>
<td><a href="name/">name</a></td>
<td>A string used as a name or title.</td>
</tr>
<tr>
<td><a href="negated/">negated</a></td>
<td>If true, the relationship between the subject and object is negated. For example, consider an association where the subject is a protein ID, the object is the GO term for "glucose biosynthesis", and the predicate is "involved in". With the "negated" field set to false, the association is interpreted as "<protein ID> is involved in glucose biosynthesis". With the "negated" field set to true, the association is interpreted as "<protein ID> is not involved in glucose biosynthesis".</td>
</tr>
<tr>
<td><a href="object/">object</a></td>
<td>Note the range of this slot is always a node. If the triple represents a literal, instead value will be populated</td>
</tr>
<tr>
<td><a href="p_value/">p_value</a></td>
<td>The 'score' of the feature. The semantics of this field are ill-defined. P-values should be used for ab initio gene prediction features.</td>
</tr>
<tr>
<td><a href="participant_type/">participant_type</a></td>
<td>The type of participant in the protocol.</td>
</tr>
<tr>
<td><a href="predicate/">predicate</a></td>
<td>The predicate of the statement</td>
</tr>
<tr>
<td><a href="prefix/">prefix</a></td>
<td>A standardized prefix such as 'GO' or 'rdf' or 'FlyBase'</td>
</tr>
<tr>
<td><a href="primary_knowledge_source/">primary_knowledge_source</a></td>
<td>The knowledge source that created the association. Should be a UUID from the DataSource table.</td>
</tr>
<tr>
<td><a href="project_id/">project_id</a></td>
<td>Internal (CDM) unique identifier for a project.</td>
</tr>
<tr>
<td><a href="protein_id/">protein_id</a></td>
<td>Internal (CDM) unique identifier for a protein.</td>
</tr>
<tr>
<td><a href="protocol_id/">protocol_id</a></td>
<td>Internal (CDM) unique identifier for a protocol.</td>
</tr>
<tr>
<td><a href="protocol_participant_id/">protocol_participant_id</a></td>
<td>The unique identifier for the protocol participant.</td>
</tr>
<tr>
<td><a href="publication_id/">publication_id</a></td>
<td>Unique identifier for a publication - e.g. PMID, DOI, URL, etc.</td>
</tr>
<tr>
<td><a href="quality/">quality</a></td>
<td>The quality of the measurement, indicating the confidence that one can have in its correctness.</td>
</tr>
<tr>
<td><a href="raw_value/">raw_value</a></td>
<td>Raw value from the source data. May or may not include units or other unstructured information.</td>
</tr>
<tr>
<td><a href="relationship/">relationship</a></td>
<td>Relationship between this identifier and the entity in the <code>entity_id</code> field.</td>
</tr>
<tr>
<td><a href="sample_id/">sample_id</a></td>
<td>Internal (CDM) unique identifier for a sample.</td>
</tr>
<tr>
<td><a href="scaf_bp/">scaf_bp</a></td>
<td>Total size in bp of all scaffolds</td>
</tr>
<tr>
<td><a href="scaf_L50/">scaf_L50</a></td>
<td>Given a set of scaffolds, the L50 is defined as the sequence length of the shortest scaffold at 50% of the total contig collection length</td>
</tr>
<tr>
<td><a href="scaf_L90/">scaf_L90</a></td>
<td>The L90 statistic is less than or equal to the L50 statistic; it is the length for which the collection of all scaffolds of that length or longer contains at least 90% of the sum of the lengths of all scaffolds.</td>
</tr>
<tr>
<td><a href="scaf_l_gt50k/">scaf_l_gt50k</a></td>
<td>The total length of scaffolds longer than 50,000 base pairs</td>
</tr>
<tr>
<td><a href="scaf_logsum/">scaf_logsum</a></td>
<td>The sum of the (length*log(length)) of all scaffolds, times some constant. Increase the contiguity, the score will increase</td>
</tr>
<tr>
<td><a href="scaf_max/">scaf_max</a></td>
<td>Maximum scaffold length</td>
</tr>
<tr>
<td><a href="scaf_N50/">scaf_N50</a></td>
<td>Given a set of scaffolds, each with its own length, the N50 count is defined as the smallest number of scaffolds whose length sum makes up half of contig collection size</td>
</tr>
<tr>
<td><a href="scaf_N90/">scaf_N90</a></td>
<td>Given a set of scaffolds, each with its own length, the N90 count is defined as the smallest number of scaffolds whose length sum makes up 90% of contig collection size</td>
</tr>
<tr>
<td><a href="scaf_n_gt50K/">scaf_n_gt50K</a></td>
<td>The number of scaffolds longer than 50,000 base pairs.</td>
</tr>
<tr>
<td><a href="scaf_pct_gt50K/">scaf_pct_gt50K</a></td>
<td>The percentage of the total assembly length represented by scaffolds longer than 50,000 base pairs</td>
</tr>
<tr>
<td><a href="scaf_powsum/">scaf_powsum</a></td>
<td>Powersum of all scaffolds is the same as logsum except that it uses the sum of (length*(length^P)) for some power P (default P=0.25).</td>
</tr>
<tr>
<td><a href="score/">score</a></td>
<td>Output from the clustering protocol indicating how closely a member matches the representative.</td>
</tr>
<tr>
<td><a href="sequence/">sequence</a></td>
<td>The protein amino acid sequence.</td>
</tr>
<tr>
<td><a href="sequence_id/">sequence_id</a></td>
<td>Internal (CDM) unique identifier for a sequence.</td>
</tr>
<tr>
<td><a href="source/">source</a></td>
<td>The source for a specific piece of information; should be a CDM internal ID of a source in the DataSource table.</td>
</tr>
<tr>
<td><a href="source_database/">source_database</a></td>
<td>ID of the data source from which this entity came.</td>
</tr>
<tr>
<td><a href="specific_ecosystem/">specific_ecosystem</a></td>
<td>JGI GOLD descriptor representing the most specific level of ecosystem categorization. May be "Unclassified".</td>
</tr>
<tr>
<td><a href="start/">start</a></td>
<td>The start and end coordinates of the feature are given in positive 1-based int coordinates, relative to the landmark given in column one. Start is always less than or equal to end. For features that cross the origin of a circular feature (e.g. most bacterial genomes, plasmids, and some viral genomes), the requirement for start to be less than or equal to end is satisfied by making end = the position of the end + the length of the landmark feature. For zero-length features, such as insertion sites, start equals end and the implied site is to the right of the indicated base in the direction of the landmark.</td>
</tr>
<tr>
<td><a href="strand/">strand</a></td>
<td>The strand of the feature.</td>
</tr>
<tr>
<td><a href="subject/">subject</a></td>
<td>The subject of the statement</td>
</tr>
<tr>
<td><a href="type/">type</a></td>
<td>The type of the entity. Should be a term from the sequence ontology.</td>
</tr>
<tr>
<td><a href="unit/">unit</a></td>
<td>The unit of the quantity. Should be a term from UCUM.</td>
</tr>
<tr>
<td><a href="updated/">updated</a></td>
<td>Date/timestamp for when the entity was updated in the CDM.</td>
</tr>
<tr>
<td><a href="url/">url</a></td>
<td>The URL from which the data was loaded.</td>
</tr>
<tr>
<td><a href="value/">value</a></td>
<td>Note the range of this slot is always a string. Only used the triple represents a literal assertion</td>
</tr>
<tr>
<td><a href="value_cv_term_id/">value_cv_term_id</a></td>
<td>If the term comes from the controlled vocabulary, the CURIE for the term. This will always be null if the text string is not from a controlled vocabulary.</td>
</tr>
<tr>
<td><a href="version/">version</a></td>
<td>For versioned data sources, the version of the dataset.</td>
</tr>
</tbody>
</table>
<h2 id="enumerations">Enumerations</h2>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="CdsPhaseType/">CdsPhaseType</a></td>
<td>For features of type CDS (coding sequence), the phase indicates where the feature begins with reference to the reading frame. The phase is one of the integers 0, 1, or 2, indicating the number of bases that should be removed from the beginning of this feature to reach the first base of the next codon.</td>
</tr>
<tr>
<td><a href="ClusterType/">ClusterType</a></td>
<td>The type of the entities in a cluster. Must be represented by a table in the CDM schema.</td>
</tr>
<tr>
<td><a href="ContigCollectionType/">ContigCollectionType</a></td>
<td>The type of the contig set; the type of the 'omics data set. Terms are taken from the Genomics Standards Consortium where possible. See the GSC checklists at  https://genomicsstandardsconsortium.github.io/mixs/ for the controlled vocabularies used.</td>
</tr>
<tr>
<td><a href="ContributorRole/">ContributorRole</a></td>
<td>The role of a contributor to a resource.</td>
</tr>
<tr>
<td><a href="ContributorType/">ContributorType</a></td>
<td>The type of contributor being represented.</td>
</tr>
<tr>
<td><a href="EntityType/">EntityType</a></td>
<td>The type of an entity. Must be represented by a table in the CDM schema.</td>
</tr>
<tr>
<td><a href="ProteinEvidenceForExistence/">ProteinEvidenceForExistence</a></td>
<td>The evidence for the existence of a biological entity. See https://www.uniprot.org/help/protein_existence and https://www.ncbi.nlm.nih.gov/genbank/evidence/.</td>
</tr>
<tr>
<td><a href="RefSeqStatusType/">RefSeqStatusType</a></td>
<td>RefSeq status codes, taken from https://www.ncbi.nlm.nih.gov/genbank/evidence/.</td>
</tr>
<tr>
<td><a href="SequenceType/">SequenceType</a></td>
<td>The type of sequence being represented.</td>
</tr>
<tr>
<td><a href="StrandType/">StrandType</a></td>
<td>The strand that a feature appears on relative to a landmark. Also encompasses unknown or irrelevant strandedness.</td>
</tr>
</tbody>
</table>
<h2 id="types">Types</h2>
<table>
<thead>
<tr>
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Boolean/">Boolean</a></td>
<td>A binary (true or false) value</td>
</tr>
<tr>
<td><a href="Curie/">Curie</a></td>
<td>a compact URI</td>
</tr>
<tr>
<td><a href="DataSourceUuid/">DataSourceUuid</a></td>
<td>A UUID that identifies a data source in the CDM.</td>
</tr>
<tr>
<td><a href="Date/">Date</a></td>
<td>a date (year, month and day) in an idealized calendar</td>
</tr>
<tr>
<td><a href="DateOrDatetime/">DateOrDatetime</a></td>
<td>Either a date or a datetime</td>
</tr>
<tr>
<td><a href="Datetime/">Datetime</a></td>
<td>The combination of a date and time</td>
</tr>
<tr>
<td><a href="Decimal/">Decimal</a></td>
<td>A real number with arbitrary precision that conforms to the xsd:decimal specification</td>
</tr>
<tr>
<td><a href="Double/">Double</a></td>
<td>A real number that conforms to the xsd:double specification</td>
</tr>
<tr>
<td><a href="Float/">Float</a></td>
<td>A real number that conforms to the xsd:float specification</td>
</tr>
<tr>
<td><a href="Integer/">Integer</a></td>
<td>An integer</td>
</tr>
<tr>
<td><a href="Iso8601/">Iso8601</a></td>
<td>A date in ISO 8601 format, e.g. 2024-04-05T12:34:56Z. "Z" indicates UTC time.</td>
</tr>
<tr>
<td><a href="Jsonpath/">Jsonpath</a></td>
<td>A string encoding a JSON Path. The value of the string MUST conform to JSON Point syntax and SHOULD dereference to zero or more valid objects within the current instance document when encoded in tree form.</td>
</tr>
<tr>
<td><a href="Jsonpointer/">Jsonpointer</a></td>
<td>A string encoding a JSON Pointer. The value of the string MUST conform to JSON Point syntax and SHOULD dereference to a valid object within the current instance document when encoded in tree form.</td>
</tr>
<tr>
<td><a href="LiteralAsStringType/">LiteralAsStringType</a></td>
<td></td>
</tr>
<tr>
<td><a href="LocalCurie/">LocalCurie</a></td>
<td>A CURIE that exists as a subject in the <code>statements</code> table (i.e. <code>Statements.subject</code>). Should not be used for external identifiers.</td>
</tr>
<tr>
<td><a href="Ncname/">Ncname</a></td>
<td>Prefix part of CURIE</td>
</tr>
<tr>
<td><a href="NodeIdType/">NodeIdType</a></td>
<td>IDs are either CURIEs, IRI, or blank nodes. IRIs are wrapped in &lt;&gt;s to distinguish them from CURIEs, but in general it is good practice to populate the [prefixes][Prefixes.md] table such that they are shortened to CURIEs. Blank nodes are ids starting with <code>_:</code>.</td>
</tr>
<tr>
<td><a href="Nodeidentifier/">Nodeidentifier</a></td>
<td>A URI, CURIE or BNODE that represents a node in a model.</td>
</tr>
<tr>
<td><a href="Objectidentifier/">Objectidentifier</a></td>
<td>A URI or CURIE that represents an object in the model.</td>
</tr>
<tr>
<td><a href="Sparqlpath/">Sparqlpath</a></td>
<td>A string encoding a SPARQL Property Path. The value of the string MUST conform to SPARQL syntax and SHOULD dereference to zero or more valid objects within the current instance document when encoded as RDF.</td>
</tr>
<tr>
<td><a href="String/">String</a></td>
<td>A character string</td>
</tr>
<tr>
<td><a href="Time/">Time</a></td>
<td>A time object represents a (local) time of day, independent of any particular day</td>
</tr>
<tr>
<td><a href="Uri/">Uri</a></td>
<td>a complete URI</td>
</tr>
<tr>
<td><a href="Uriorcurie/">Uriorcurie</a></td>
<td>a URI or a CURIE</td>
</tr>
<tr>
<td><a href="UUID/">UUID</a></td>
<td>A universally unique ID, generated using uuid4, with the prefix "CDM:".</td>
</tr>
</tbody>
</table>
<h2 id="subsets">Subsets</h2>
<table>
<thead>
<tr>
<th>Subset</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td></td>
</tr>
</tbody>
</table>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": ".", "features": ["content.tabs.link"], "search": "assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="assets/javascripts/bundle.92b07e13.min.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="javascripts/tablesort.js"></script>
      
    
  </body>
</html>